zh.json

```diff
@@ -1,3770 +1 @@
24.Text","id":"Case History","str":"案件历史"},{"key":"8634768E4D6C9A9C68755EA705B1A09E","location":"/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_25.Text","id":"Box","str":"盒子"},{"key":"8772BC8B4A8B67D8026B5D806054E488","location":"/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_27.Text","id":"Sell All","str":"全部出售"},{"key":"F5A1B20A4E970FD062976FB28CEBCEC2","location":"/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_28.Text","id":"Reset Career Mode","str":"重置职业模式"},{"key":"5A417E174DAE0F1865DA91B85DC6AFF4","location":"/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_5.Text","id":"Buy All","str":"全部购买"},{"key":"F703F19542906B2F29A58E83811DB3AD","location":"/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_7.Text","id":"You don't have enough money","str":"金钱不足"},{"key":"84B3B484489A7A0017B755B62F30FC82","location":"/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_8.Text","id":"Accept Case","str":"接下案件"},{"key":"A0A8C36D40881280C500F9A2E5AFFDAB","location":"/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_9.Text","id":"Location","str":"地点"},{"key":"5981A77041F835414316E0BEA0C3C8F8","location":"/Game/UI/career/caseItem.caseItem_C:GetText [Script Bytecode]","id":"YOU WIN","str":"你赢了"},{"key":"CCDA7BC64A6EA8E1824143B0E488DCDA","location":"/Game/UI/career/caseItem.caseItem_C:GetText [Script Bytecode]","id":"Dropped","str":"已丢弃"},{"key":"FE160B74470133E4B513D18D4566D063","location":"/Game/UI/career/caseItem.caseItem_C:GetText [Script Bytecode]","id":"YOU LOSE","str":"你输了"},{"key":"B8F169E14DBF7CFF567A468DBF711F46","location":"/Game/UI/career/caseItem.caseItem_C:GetText_2 [Script Bytecode]","id":"{count} days","str":"{count}天"},{"key":"D3C6D63B4A98961E44BEEB8EC2FEF2D1","location":"/Game/UI/career/chestUI.chestUI_C:ExecuteUbergraph_chestUI [Script Bytecode]","id":"Some of the elements you already have equipped","str":"你已经装备了一些元素"},{"key":"E70BD9AB4A85E6A6ADCE4B8E7C2F2260","location":"/Game/UI/career/chestUI.chestUI_C:ExecuteUbergraph_chestUI [Script Bytecode]","id":"You can only equip a maximum of 10 items","str":"最多只能装备10件物品"},{"key":"F4678833417F67BF4FB742B959AD7E80","location":"/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock.Text","id":"Only items that can be equipped will appear","str":"只有可装备的物品才会出现"},{"key":"02384976426960372C5EAE9FE9459154","location":"/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock_2.Text","id":"Your inventory","str":"你的库存"},{"key":"AE6D75704E0C22E2F8C107A8F12A0222","location":"/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock_3.Text","id":"Your equiped items","str":"你的装备物品"},{"key":"F880818144C667A994BEEF9200987EF4","location":"/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock_6.Text","id":"Chest","str":"宝箱"},{"key":"C5BF8D15413D2BD751F76EBE5D96E205","location":"/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock_9.Text","id":"All items placed in the chest will disappear when you exit the map.","str":"当你退出地图时，放置在宝箱中的所有物品都会消失。"},{"key":"522202404C57
```

Comment:
```suggestion
[
    {
        "key": "8C6EFCE649B93D5275B158901F4CAB7A",
        "location": "/Game/Blueprints/driveVan.driveVan_C:ExecuteUbergraph_driveVan [Script Bytecode]",
        "id": "Back to the safehouse",
        "str": "回到安全屋"
    },
    {
        "key": "E0A26C2A477073293C2A5093569FFE5E",
        "location": "/Game/Blueprints/driveVan.driveVan_C:ExecuteUbergraph_driveVan [Script Bytecode]",
        "id": "If you return to the safehouse, the house will restore its state and you will lose the equipped items",
        "str": "如果你回到安全屋，房子将恢复原状，你将失去装备"
    },
    {
        "key": "C312729245B39D3E05385886891922CF",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(0).texts",
        "id": "God, Save Us",
        "str": "上帝，救救我们"
    },
    {
        "key": "78CDF9F24D7F05B20507E99EF8B9D6B4",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(1).texts",
        "id": "Where is the hope?",
        "str": "希望在何处？"
    },
    {
        "key": "806D1B0C4EFCAD5D435AD6A24A208010",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(10).texts",
        "id": "Who will listen to us?",
        "str": "谁会听我们的？"
    },
    {
        "key": "4CC12ED04C0CCDD4B99137BA4EFF21F1",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(2).texts",
        "id": "Everything is falling apart",
        "str": "一切正在分崩离析"
    },
    {
        "key": "D3B042044111E5CC7AE8239FB511F147",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(3).texts",
        "id": "Darkness surrounds us",
        "str": "黑暗包围着我们"
    },
    {
        "key": "46CE50D24305DF8EE5E3069A686AD62E",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(4).texts",
        "id": "What do we have left?",
        "str": "我们还剩下什么？"
    },
    {
        "key": "7C103CB840F8E092D02937A8D8ACCCCF",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(5).texts",
        "id": "Nothing makes sense anymore",
        "str": "毫无意义"
    },
    {
        "key": "C868184542C8CF8CE45E35881EB2FB7F",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(6).texts",
        "id": "The pain is overwhelming",
        "str": "痛彻心扉"
    },
    {
        "key": "1B91C6234DA146AEDD1F52A4882D4BE0",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(7).texts",
        "id": "There seems to be no way out",
        "str": "似乎没有出路"
    },
    {
        "key": "8013788A45FE0968E12AA3BAF0AE5233",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(8).texts",
        "id": "The silence is deafening",
        "str": "震耳欲聋的沉默"
    },
    {
        "key": "FB77087A4862A67B758D529F87748C2F",
        "location": "/Game/Blueprints/dynamicActions/daycare/blackboard.Default__blackboard_C.texts(9).texts",
        "id": "Chaos consumes us",
        "str": "混沌吞噬我们"
    },
    {
        "key": "1C4A0DD846CCF542A5D75C843FB42C43",
        "location": "/Game/Blueprints/dynamicDoll.dynamicDoll_C:ExecuteUbergraph_dynamicDoll [Script Bytecode]",
        "id": "This doll is under effects, wait for the effect to finish to apply another one",
        "str": "这个人偶正受到某种效果的影响，等待效果结束才能施加另一个。"
    },
    {
        "key": "5DAA61584F29C51BE364D59FAF17F7CB",
        "location": "/Game/Blueprints/dynamicDoll.dynamicDoll_C:ExecuteUbergraph_dynamicDoll [Script Bytecode]",
        "id": "You need to use the book of souls before you save the imposter doll",
        "str": "你得先用灵魂之书才能收集卧底人偶"
    },
    {
        "key": "073BF882481ECDE57F50FF98957A3C22",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:checkPopUps [Script Bytecode]",
        "id": "You've unlocked a new difficulty!",
        "str": "你已解锁新难度！"
    },
    {
        "key": "51B58C89473E1AE2A37E6A8118DC37C1",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:checkPopUps [Script Bytecode]",
        "id": "Have you ever wondered where the information about the dolls comes from? In career mode, you will discover the information about the dolls with infinite cases, economy, levels, shop, crafting, and much more!",
        "str": "你是否想过人偶的信息是从哪里来的？在职业模式中，你将通过无限案例、经济、等级、商店、制作等等来发现关于人偶的信息！"
    },
    {
        "key": "839DA89C42E027A3CA515CA3BC5BE204",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:checkPopUps [Script Bytecode]",
        "id": "You can try it now in PC",
        "str": "你现在可以用电脑进行尝试"
    },
    {
        "key": "9472CB7546EB19A64D00B9B40CF7BC7D",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:checkPopUps [Script Bytecode]",
        "id": "You've unlocked a new extra mode!",
        "str": "你解锁了新的额外模式！"
    },
    {
        "key": "B372B7564467D935DC5122B7415A738C",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:checkPopUps [Script Bytecode]",
        "id": "You've unlocked the career mode!",
        "str": "你已解锁职业模式！"
    },
    {
        "key": "FE11E6ED4FA63C5B12631383E56A20B5",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:checkPopUps [Script Bytecode]",
        "id": "You've unlocked the {map} Map!",
        "str": "你解锁了{map}地图！"
    },
    {
        "key": "8098B1CA4AED33E029A8D4878EE21A49",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:ExecuteUbergraph_HorrorEngineGameMode [Script Bytecode]",
        "id": "You've broken your new record with {new} nights! of the {old} nights before!",
        "str": "你以{new}夜的成绩，打破了先前{old}夜的记录！"
    },
    {
        "key": "B4559452445FD99D5366B68268BCFC9E",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:ExecuteUbergraph_HorrorEngineGameMode [Script Bytecode]",
        "id": "Congratulations!",
        "str": "恭喜！"
    },
    {
        "key": "F83E76BC49FC84244322C382556B6E5F",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:ExecuteUbergraph_HorrorEngineGameMode [Script Bytecode]",
        "id": "Oh no...",
        "str": "哦不……"
    },
    {
        "key": "FC67115746EDB65A022A51B4C1176418",
        "location": "/Game/Blueprints/HorrorEngineGameMode.HorrorEngineGameMode_C:ExecuteUbergraph_HorrorEngineGameMode [Script Bytecode]",
        "id": "You've got {new} nights, you haven't managed to beat your record of {old} nights",
        "str": "你度过了{new}晚，没能打破之前的{old}晚记录"
    },
    {
        "key": "27228C9F429D8354B0534FB17BD564AA",
        "location": "/Game/Blueprints/interactableUI3D.interactableUI3D_C:ExecuteUbergraph_interactableUI3D [Script Bytecode]",
        "id": "Interact",
        "str": "相互影响"
    },
    {
        "key": "0CA5844D4070A29607D91D9C17419FCD",
        "location": "/Game/Blueprints/options.options_C:ExecuteUbergraph_options [Script Bytecode]",
        "id": "Skip all hours",
        "str": "跳过所有时分"
    },
    {
        "key": "34568E7B486905A5400643A88FA2E8A8",
        "location": "/Game/Blueprints/options.options_C:ExecuteUbergraph_options [Script Bytecode]",
        "id": "Show list",
        "str": "显示列表"
    },
    {
        "key": "8048C27C4B137230FE4708ADFD1AD7F4",
        "location": "/Game/Blueprints/options.options_C:ExecuteUbergraph_options [Script Bytecode]",
        "id": "Show Jumpscares",
        "str": "显示惊吓"
    },
    {
        "key": "BCA4F7E9472918A4030087B0CBCF7608",
        "location": "/Game/Blueprints/PC.PC_C:ExecuteUbergraph_PC [Script Bytecode]",
        "id": "OK",
        "str": "确定"
    },
    {
        "key": "79B48418402866A435B0658D650C041D",
        "location": "/Game/Blueprints/weaponInteract.weaponInteract_C:ExecuteUbergraph_weaponInteract [Script Bytecode]",
        "id": "You've used everything you needed to use here.",
        "str": "你用完了这里所需的全部东西。"
    },
    {
        "key": "8E794D794CCEA0D30D037D8DA2BFD248",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Lights",
        "str": "灯光"
    },
    {
        "key": "B41DE7E344BBE9E3F4ED828B3AC5DE1A",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow.sections_22_4031E9264112E4E86BC48BA9D729A311(0).sections_22_4031E9264112E4E86BC48BA9D729A311.section_3_07527C2F4BE4A70097D1ACA12D7EA7DE",
        "id": "Color",
        "str": "颜色"
    },
    {
        "key": "3D3092E04B30FAE9F91970831F869B44",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_0.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Wall",
        "str": "墙壁"
    },
    {
        "key": "E93B4EC241D67CCBA288EAB44234C576",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_0.sections_22_4031E9264112E4E86BC48BA9D729A311(0).sections_22_4031E9264112E4E86BC48BA9D729A311.section_3_07527C2F4BE4A70097D1ACA12D7EA7DE",
        "id": "Material",
        "str": "材料"
    },
    {
        "key": "31D983AD40E7D0F0976B7CAC11BD3D12",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_1.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Roof",
        "str": "屋顶"
    },
    {
        "key": "54DD43104162D7A89F5B2ABADBA7F0A6",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_14.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Van Lights",
        "str": "面包车灯光"
    },
    {
        "key": "F1D636544DE5ADC7945E9789ACC63D6C",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_15.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Van wall",
        "str": "面包车墙壁"
    },
    {
        "key": "F398437D44CE91CF11D919A1208038ED",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_15.sections_22_4031E9264112E4E86BC48BA9D729A311(2).sections_22_4031E9264112E4E86BC48BA9D729A311.section_3_07527C2F4BE4A70097D1ACA12D7EA7DE",
        "id": "Size",
        "str": "大小"
    },
    {
        "key": "805F56F9484DE7ED01BC83B1E3304EE7",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_16.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Van floor",
        "str": "面包车底板"
    },
    {
        "key": "BEF09CA34BAE3DC7635E8485BB667E28",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_17.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Van exterior",
        "str": "面包车外部"
    },
    {
        "key": "4BA592A54AE29F2B8AA5DCB59A988A0F",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_18.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Van Hanging doll",
        "str": "面包车吊挂人偶"
    },
    {
        "key": "7D2E931045A6E1506DEEC6BD2C264ED3",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_19.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Van Left Side",
        "str": "面包车左侧"
    },
    {
        "key": "109D973D484D997685448980DCB79533",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_2.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Floor",
        "str": "地板"
    },
    {
        "key": "C539F07A44FA320908B87FADABCFB0C9",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_20.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Van Right Side",
        "str": "面包车右侧"
    },
    {
        "key": "164553EC4B6A9D265430F3B6A25830E5",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_21.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Van Bed",
        "str": "面包车床"
    },
    {
        "key": "8D62136643D4EAA319AD22BBFD4C8FD1",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_3.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Statue",
        "str": "雕像"
    },
    {
        "key": "6A1B28C14FD17FB8222E60837327DA35",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_3.sections_22_4031E9264112E4E86BC48BA9D729A311(0).sections_22_4031E9264112E4E86BC48BA9D729A311.section_3_07527C2F4BE4A70097D1ACA12D7EA7DE",
        "id": "Object",
        "str": "物体"
    },
    {
        "key": "E7D75C9A40FB9B41721A46B5B7198EF1",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_3.sections_22_4031E9264112E4E86BC48BA9D729A311(1).sections_22_4031E9264112E4E86BC48BA9D729A311.section_3_07527C2F4BE4A70097D1ACA12D7EA7DE",
        "id": "Sticker",
        "str": "贴纸"
    },
    {
        "key": "A1DAA0264F4E98D0E7FF1E89CCEF4110",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_4.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Left Place",
        "str": "左侧位置"
    },
    {
        "key": "D2EAFC9348E348C7516300BF122B9702",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_5.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Right Place",
        "str": "右侧位置"
    },
    {
        "key": "2F29E9E841620DC6B54893A7BE60FE77",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_6.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "Desktop",
        "str": "桌面"
    },
    {
        "key": "7D39CA6640F288A9002846AE7D7F4041",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_6.sections_22_4031E9264112E4E86BC48BA9D729A311(0).sections_22_4031E9264112E4E86BC48BA9D729A311.section_3_07527C2F4BE4A70097D1ACA12D7EA7DE",
        "id": "Mini Object",
        "str": "迷你物体"
    },
    {
        "key": "0B88630D44BEE72FF2A76095115344A8",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_6.sections_22_4031E9264112E4E86BC48BA9D729A311(1).sections_22_4031E9264112E4E86BC48BA9D729A311.section_3_07527C2F4BE4A70097D1ACA12D7EA7DE",
        "id": "Doll Object",
        "str": "人偶物体"
    },
    {
        "key": "7095C09D4014D1FB9E4C1D9529586155",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_7.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "PC Left Side",
        "str": "电脑左侧"
    },
    {
        "key": "1AEE02D7485216A7EBEC93AD95D81A25",
        "location": "/Game/data-tables/customizerRooms.customizerRooms.NewRow_8.label_17_E97135F84728B1D3E2D91DAE099E4D25",
        "id": "PC Right Side",
        "str": "电脑右侧"
    },
    {
        "key": "D249146742516BB46178BA859A421873",
        "location": "/Game/data-tables/itemList.itemList.NewRow.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Take doll photos for fill the clues",
        "str": "拍摄人偶照片填补线索"
    },
    {
        "key": "69F832714D668740651DED8EBBD4EBB8",
        "location": "/Game/data-tables/itemList.itemList.NewRow.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Camera",
        "str": "相机"
    },
    {
        "key": "5471B2664F74874D8556F9B63E352A84",
        "location": "/Game/data-tables/itemList.itemList.NewRow_1.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Take flashlight",
        "str": "拿取手电筒"
    },
    {
        "key": "4020C7B84456D9F178DC488EAB546977",
        "location": "/Game/data-tables/itemList.itemList.NewRow_1.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Flashlight",
        "str": "手电筒"
    },
    {
        "key": "93917B7240A1743F582D7E9208B16F62",
        "location": "/Game/data-tables/itemList.itemList.NewRow_10.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Paper that reveals dislikes",
        "str": "揭示厌恶的纸"
    },
    {
        "key": "2E06BFD64361FE191A01DD86C96EBAEE",
        "location": "/Game/data-tables/itemList.itemList.NewRow_10.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Soul Dark Paper",
        "str": "灵魂暗纸"
    },
    {
        "key": "AE174EC646F0BB590D3A1CAB82B6AC70",
        "location": "/Game/data-tables/itemList.itemList.NewRow_11.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A small, dark, and shriveled heart.",
        "str": "一颗小小的阴暗枯萎之心。"
    },
    {
        "key": "F63BB439446A93B294D5789005AB927F",
        "location": "/Game/data-tables/itemList.itemList.NewRow_11.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Crow's Heart",
        "str": "乌鸦之心"
    },
    {
        "key": "3B43B2384C4FFE5DE5823BAB1338EC37",
        "location": "/Game/data-tables/itemList.itemList.NewRow_12.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "An ancient piece of paper.",
        "str": "古老的纸张。"
    },
    {
        "key": "CEB55E1E480248C3E1553AB0E3E06BA3",
        "location": "/Game/data-tables/itemList.itemList.NewRow_12.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Parchment",
        "str": "羊皮纸"
    },
    {
        "key": "743C50254A73CE29F7B3D7858FFBA46F",
        "location": "/Game/data-tables/itemList.itemList.NewRow_13.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Dark and thick liquid.",
        "str": "黑暗且浓稠的液体"
    },
    {
        "key": "35FE04294F60A661E2A93E8CF5BAB2C9",
        "location": "/Game/data-tables/itemList.itemList.NewRow_13.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Snake's Blood",
        "str": "蛇血"
    },
    {
        "key": "1D8C5BFB408218C11B5F8E88DED5DDE0",
        "location": "/Game/data-tables/itemList.itemList.NewRow_14.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Large and grayish feather.",
        "str": "浅灰色的大羽毛。"
    },
    {
        "key": "FD5D53AB4A626CFD46CF4E93053F9719",
        "location": "/Game/data-tables/itemList.itemList.NewRow_14.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Vulture's feather",
        "str": "秃鹫的羽毛"
    },
    {
        "key": "14AD66ED4AA7704FAEA0CD892A4DDD54",
        "location": "/Game/data-tables/itemList.itemList.NewRow_15.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A white flower with soft petals.",
        "str": "一朵花瓣柔软的白花。"
    },
    {
        "key": "6F62E8EE4D6306627A7D92B69CABCC13",
        "location": "/Game/data-tables/itemList.itemList.NewRow_15.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Lily flower",
        "str": "百合花"
    },
    {
        "key": "C28723EC476B280B7C783CAAD9FE2784",
        "location": "/Game/data-tables/itemList.itemList.NewRow_16.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A white and delicate feather.",
        "str": "脆弱的白色羽毛。"
    },
    {
        "key": "814D9929402ED9670B11B8A5F77DD89E",
        "location": "/Game/data-tables/itemList.itemList.NewRow_16.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Dove's feather",
        "str": "鸽子的羽毛"
    },
    {
        "key": "4E005C71470C8416B3B419B0E3E4CD6E",
        "location": "/Game/data-tables/itemList.itemList.NewRow_17.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A clear and pure crystal.",
        "str": "清澈纯净的水晶。"
    },
    {
        "key": "34549E97462027359C04D69FE1AC1E77",
        "location": "/Game/data-tables/itemList.itemList.NewRow_17.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Quartz crystal",
        "str": "石英晶体"
    },
    {
        "key": "821C250E485ED0C26041C8830F8E0625",
        "location": "/Game/data-tables/itemList.itemList.NewRow_18.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A durable and flexible material.",
        "str": "耐用的柔韧材料。"
    },
    {
        "key": "008C17DA426E713F7F3A418B4FDE1BE4",
        "location": "/Game/data-tables/itemList.itemList.NewRow_18.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Leather",
        "str": "皮革"
    },
    {
        "key": "F746170F4C38F9AE155525AEA0B83F1C",
        "location": "/Game/data-tables/itemList.itemList.NewRow_19.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Soft and fragrant rose petals.",
        "str": "柔软芬芳的玫瑰花瓣。"
    },
    {
        "key": "AF3764E04E32829AB3E75FBD938BF9C1",
        "location": "/Game/data-tables/itemList.itemList.NewRow_19.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Rose Petals",
        "str": "玫瑰花瓣"
    },
    {
        "key": "ADEEDBE04D10D55BE7B838934AC45E6C",
        "location": "/Game/data-tables/itemList.itemList.NewRow_2.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Activate the final battle to uncover the impostor.",
        "str": "激活最后的战斗，揭穿卧底伪装。"
    },
    {
        "key": "CA5357694E9708D4AC84699C2DC77CBD",
        "location": "/Game/data-tables/itemList.itemList.NewRow_2.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Soul Book",
        "str": "灵魂之书"
    },
    {
        "key": "F2BAB4874C2FDD509DA5CDA67FF91E37",
        "location": "/Game/data-tables/itemList.itemList.NewRow_20.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A golden and sweet liquid.",
        "str": "味甜的金色液体。"
    },
    {
        "key": "72400A564910846D7B2ECEAD2631331D",
        "location": "/Game/data-tables/itemList.itemList.NewRow_20.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Honey",
        "str": "蜂蜜"
    },
    {
        "key": "0A4EE34344113573CE97F6B1E7151DDE",
        "location": "/Game/data-tables/itemList.itemList.NewRow_21.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A fragrant vanilla pod.",
        "str": "一颗散发着香气的香草豆。"
    },
    {
        "key": "DDB9D7494B887F84ABF6FA9C68A57B95",
        "location": "/Game/data-tables/itemList.itemList.NewRow_21.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Vanilla",
        "str": "香草"
    },
    {
        "key": "9CAB638548874B5E9D71628F2EE226D1",
        "location": "/Game/data-tables/itemList.itemList.NewRow_22.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A transparent and elegant container.",
        "str": "透明的优雅容器。"
    },
    {
        "key": "8D70CD774E635393573985830A5099B8",
        "location": "/Game/data-tables/itemList.itemList.NewRow_22.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Crystal Bottle",
        "str": "水晶瓶"
    },
    {
        "key": "653072BC4D6CCFF0D204B6B4887A557F",
        "location": "/Game/data-tables/itemList.itemList.NewRow_23.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Dark and pungent bulb.",
        "str": "气味浓厚的暗灯泡。"
    },
    {
        "key": "20C95AA345D28767010999ADA95FBAE7",
        "location": "/Game/data-tables/itemList.itemList.NewRow_23.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Black Garlic",
        "str": "黑蒜"
    },
    {
        "key": "F249EDB640DA6D9A4C0334BE6E3B5DAB",
        "location": "/Game/data-tables/itemList.itemList.NewRow_24.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A bitter and viscous liquid.",
        "str": "味苦的粘稠液体。"
    },
    {
        "key": "F3539C0B4A4D262BEEADFCBC91E58848",
        "location": "/Game/data-tables/itemList.itemList.NewRow_24.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Toad Bile",
        "str": "蟾蜍胆汁"
    },
    {
        "key": "8B556C7E4DB153BCF7AF4AA0042DBADC",
        "location": "/Game/data-tables/itemList.itemList.NewRow_25.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Green and stinging leaves.",
        "str": "棘刺绿叶。"
    },
    {
        "key": "F5FA5A154002D6C4EDB30FB45794A50A",
        "location": "/Game/data-tables/itemList.itemList.NewRow_25.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Nettle",
        "str": "荨麻"
    },
    {
        "key": "E67B208049756CC3B5F16089AC41B470",
        "location": "/Game/data-tables/itemList.itemList.NewRow_26.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Refreshing green leaves.",
        "str": "清新的绿叶。"
    },
    {
        "key": "AC46E8C5498572C242CBADA69A94A0A7",
        "location": "/Game/data-tables/itemList.itemList.NewRow_26.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Mint",
        "str": "薄荷"
    },
    {
        "key": "FD27A6D94DD43A861953BA81606B6B11",
        "location": "/Game/data-tables/itemList.itemList.NewRow_27.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A fragrant cinnamon stick.",
        "str": "芳香的肉桂棒。"
    },
    {
        "key": "01C8EACC48F42AB3BC9A26B9A45C3E10",
        "location": "/Game/data-tables/itemList.itemList.NewRow_27.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Cinnamon",
        "str": "肉桂"
    },
    {
        "key": "6FDA4FB448EF65CB05B874842143CD74",
        "location": "/Game/data-tables/itemList.itemList.NewRow_28.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Small and juicy fruits.",
        "str": "多汁的小型水果。"
    },
    {
        "key": "091617E74F451B4492E4B78A534E9BF3",
        "location": "/Game/data-tables/itemList.itemList.NewRow_28.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Berries",
        "str": "浆果"
    },
    {
        "key": "24B7D6584C3AF4730B3365BB46DDA130",
        "location": "/Game/data-tables/itemList.itemList.NewRow_29.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Dark and toxic leaves.",
        "str": "有毒的暗色叶子。"
    },
    {
        "key": "4852DFEF4B35409C425DA1B5468B83D9",
        "location": "/Game/data-tables/itemList.itemList.NewRow_29.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Belladonna leaves",
        "str": "颠茄叶"
    },
    {
        "key": "D597FBE342A7B8B23676A3BB4B81D863",
        "location": "/Game/data-tables/itemList.itemList.NewRow_3.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Take notes on the behavior of the dolls",
        "str": "记下人偶的行为"
    },
    {
        "key": "ED2F4CB1487E0D6D9070899567BB98B0",
        "location": "/Game/data-tables/itemList.itemList.NewRow_3.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Notebook",
        "str": "笔记本"
    },
    {
        "key": "F9560AD34A829952AE5F5498756E174C",
        "location": "/Game/data-tables/itemList.itemList.NewRow_30.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Clear and lethal liquid.",
        "str": "致命的透明液体。"
    },
    {
        "key": "32F257464B93572E3D0BF293BA036023",
        "location": "/Game/data-tables/itemList.itemList.NewRow_30.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Scorpion Venom",
        "str": "蝎毒"
    },
    {
        "key": "17BF9A32484E32062DAF9F8542DD97A7",
        "location": "/Game/data-tables/itemList.itemList.NewRow_31.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A gnarled and sinister root.",
        "str": "扭曲狰狞的树根。"
    },
    {
        "key": "F671810346E2F778C0E4E7BA69FE7D9C",
        "location": "/Game/data-tables/itemList.itemList.NewRow_31.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Mandrake Root",
        "str": "曼德拉草根"
    },
    {
        "key": "C6C0EE3547F1545A84D8268AC0025132",
        "location": "/Game/data-tables/itemList.itemList.NewRow_32.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Portable device that produces a flame to start a fire.",
        "str": "可以产生火焰的便携点火装置。"
    },
    {
        "key": "AB25CB294CD7BEF8559DFA89B96B8F9B",
        "location": "/Game/data-tables/itemList.itemList.NewRow_32.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Lighter",
        "str": "打火机"
    },
    {
        "key": "3B56E6BC4AEDD9945911B29AC8342F9D",
        "location": "/Game/data-tables/itemList.itemList.NewRow_33.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Spacious bag",
        "str": "宽敞的袋子"
    },
    {
        "key": "9412FB8146653EECA5DBB2BD2504B413",
        "location": "/Game/data-tables/itemList.itemList.NewRow_33.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Bag",
        "str": "袋子"
    },
    {
        "key": "BF1AC59D4D99C0F9E92606B3C78A909C",
        "location": "/Game/data-tables/itemList.itemList.NewRow_34.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A precious gemstone known for its extreme hardness and brilliance, commonly used in jewelry and valued for its purity and rarity.",
        "str": "一种以极高硬度和光辉而闻名的宝石，常用于珠宝，并因其纯净和稀有而被珍视。"
    },
    {
        "key": "6BF7F5344FD96A246EE841869E9C0DB0",
        "location": "/Game/data-tables/itemList.itemList.NewRow_34.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Diamond",
        "str": "钻石"
    },
    {
        "key": "C4B2BB4B48FA324D2BC7979577E55027",
        "location": "/Game/data-tables/itemList.itemList.NewRow_35.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A solid block of refined gold, highly pure, used as an investment or store of value, characterized by its weight and golden luster.",
        "str": "一个由精炼黄金制成的固体块，纯度极高，用作投资或储值，特点是其重量和金色光泽。"
    },
    {
        "key": "22B8C07045CAEA974D03AD9C0D933A4E",
        "location": "/Game/data-tables/itemList.itemList.NewRow_35.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Gold Bar",
        "str": "金条"
    },
    {
        "key": "415807E5403F54FCC16EEA9338CF267F",
        "location": "/Game/data-tables/itemList.itemList.NewRow_36.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "A trophy in the shape of a cup, usually golden, awarded as a symbol of victory in sports competitions or prestigious ceremonies.",
        "str": "一个杯形奖杯，通常是金色的，作为体育比赛或重要仪式的胜利象征而颁发。"
    },
    {
        "key": "9A3A87B047A1CC933C4F6197811B657B",
        "location": "/Game/data-tables/itemList.itemList.NewRow_36.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Gold Cup",
        "str": "金杯"
    },
    {
        "key": "86BE55834614BDEE6BACCD821A069F89",
        "location": "/Game/data-tables/itemList.itemList.NewRow_37.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "An elegant piece of jewelry made up of a string of round pearls, known for their soft luster and associated with sophistication and luxury.",
        "str": "由一串圆形珍珠组成的优雅珠宝，以其柔和的光泽著称，并与精致和奢华联系在一起。"
    },
    {
        "key": "82B77C5A456740B0D72910B21AF4A463",
        "location": "/Game/data-tables/itemList.itemList.NewRow_37.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Pearl Necklace",
        "str": "珍珠项链"
    },
    {
        "key": "C0FD3F65464661A0D220FFAF9A25C2D4",
        "location": "/Game/data-tables/itemList.itemList.NewRow_4.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Fuse to activate electrical circuits",
        "str": "启动电路的保险丝"
    },
    {
        "key": "DE011C1944710509D6330C8C62491DBA",
        "location": "/Game/data-tables/itemList.itemList.NewRow_4.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Fuse",
        "str": "保险丝"
    },
    {
        "key": "8841057C4E79D1D004AE35BDDA5930D6",
        "location": "/Game/data-tables/itemList.itemList.NewRow_5.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Elixir that reveals hatred",
        "str": "揭示厌恶的灵药"
    },
    {
        "key": "B377188C494F5DA03FE7E8A6C86AB940",
        "location": "/Game/data-tables/itemList.itemList.NewRow_6.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Elixir that reveals friendship",
        "str": "揭示友谊的灵药"
    },
    {
        "key": "7EB47B9449369EB3E3F18C972CC18786",
        "location": "/Game/data-tables/itemList.itemList.NewRow_7.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Elixir that reveals enmity",
        "str": "揭示敌意的灵药"
    },
    {
        "key": "58DD772D4ABE943A14B03D9A569701B6",
        "location": "/Game/data-tables/itemList.itemList.NewRow_8.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Elixir that reveals love",
        "str": "揭示爱情的灵药"
    },
    {
        "key": "92CDFBB847F726D64FF6169E8B2A6631",
        "location": "/Game/data-tables/itemList.itemList.NewRow_9.info_16_4DFA58E54C75086154A8DC9A319FD10E",
        "id": "Paper that reveals likes",
        "str": "揭示喜好的纸"
    },
    {
        "key": "FF45EB7F42E49E1D9D364DAFAB0467BE",
        "location": "/Game/data-tables/itemList.itemList.NewRow_9.name_12_D217604B46FF9197C7004FBE2050F6FD",
        "id": "Soul Paper",
        "str": "灵魂纸"
    },
    {
        "key": "D955FAE74433E108EB3C078EF42B2DC8",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(0).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Quiet",
        "str": "沉寂者"
    },
    {
        "key": "B53AF9564736AB6B4D8F2C81FD9CE5F6",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(1).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Unnoticed",
        "str": "无形者"
    },
    {
        "key": "4F13E4CD4029EC13370EF7B54BCF3D90",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(10).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Governor",
        "str": "支配者"
    },
    {
        "key": "740E2993476C6AB0DEDC3BA514B0FF5F",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(11).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Comedian",
        "str": "喜剧演员"
    },
    {
        "key": "8B7A2813470D5B634D42F1B18FD1C975",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(12).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Acrobat",
        "str": "杂技演员"
    },
    {
        "key": "75A0760B4CF756F7572751980578A7E4",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(13).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Single",
        "str": "单身者"
    },
    {
        "key": "D618A4A346B0EA7CBABE00AA94C80485",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(14).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Count",
        "str": "伯爵"
    },
    {
        "key": "B46005B1494B233FC63D6C92DB922873",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(15).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Doorman",
        "str": "守门人"
    },
    {
        "key": "CF3A6ECB45A9488C10D246A3FD143977",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(16).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Weasel",
        "str": "狡猾者"
    },
    {
        "key": "143BBE1843E5BD9D98FB8C9B4041B306",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(17).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Beautiful",
        "str": "美丽佳人"
    },
    {
        "key": "DFCF9FFA438E6B63C7745586401C14BA",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(18).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The New",
        "str": "新人"
    },
    {
        "key": "D8CB256A4FC1286A3BD465B95B7398D6",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(19).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Flirt",
        "str": "调情者"
    },
    {
        "key": "1D5965BE4639794BCFFC6BA942EC1A62",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(2).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Gentleman",
        "str": "绅士"
    },
    {
        "key": "73AD26AC4F1BCAD5E64E5E889E9D9A79",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(3).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Traitor",
        "str": "叛徒"
    },
    {
        "key": "996579D246DE71F2608BB5BF8C353DA4",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(4).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Kid",
        "str": "小孩"
    },
    {
        "key": "0375A2D542A72CFAD4E4CA8A30C56CC4",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(5).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Duchess",
        "str": "女公爵"
    },
    {
        "key": "FA4E21624A2D966B1B2ED19005131257",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(6).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Traditional",
        "str": "传统派"
    },
    {
        "key": "D03A184741DBDC2985FBC9A454F19494",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(7).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Nervous",
        "str": "焦躁者"
    },
    {
        "key": "60747A674F244C50BD7C42986DD084AD",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(8).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Widow",
        "str": "寡妇"
    },
    {
        "key": "554BB9D046155183D4DCF7A14F96171E",
        "location": "/Game/data-tables/mapList.mapList.NewRow.dolls_9_7FEB33284744601C3269539AAFDB6152(9).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Duke",
        "str": "公爵"
    },
    {
        "key": "F28247BF4A488DEE853285BA1EBC6188",
        "location": "/Game/data-tables/mapList.mapList.NewRow_0.name_18_82096130492E34A890A0138C4910D929",
        "id": "House",
        "str": "房屋"
    },
    {
        "key": "B4AEF1D44E3C8E5E66BF539EAD22F342",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(0).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Terrifying",
        "str": "恐怖者"
    },
    {
        "key": "C8A00764485261B746BFEE94C58D6ECA",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(1).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Scary",
        "str": "惊悚人偶"
    },
    {
        "key": "493B1FBE4D84E9D9798F7D8AAD0CF937",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(10).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Ominous",
        "str": "不祥之兆"
    },
    {
        "key": "DAFC2C134E48E1FBE3399C9D15E37173",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(11).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Unsettling",
        "str": "战栗之源"
    },
    {
        "key": "F22CBEE0429B4D716137DE9A58D8A371",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(12).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Frightening",
        "str": "骇人者"
    },
    {
        "key": "D0B2C8BA4674F0C2C3F6CAB676B10CBF",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(13).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Spooky",
        "str": "阴森者"
    },
    {
        "key": "966BA2A948A2BC025A15EBA7AD8767E0",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(14).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Diabolic",
        "str": "魔鬼"
    },
    {
        "key": "CE5521734152BB8FDF36949D84AA7F39",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(15).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Ghastly",
        "str": "惨白幽魂"
    },
    {
        "key": "7461839E43289FBE953BE6A6CD4D28D8",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(16).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Deadly",
        "str": "致命者"
    },
    {
        "key": "A31A514F4934E454FDB9F4BF3A480C84",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(17).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Grim",
        "str": "残酷者"
    },
    {
        "key": "6B78310947BA56A4D0322EAFBC650458",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(18).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Macabre",
        "str": "阴森恐怖"
    },
    {
        "key": "B98D2AB7447B5DBEA44D1D8E675432D3",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(19).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Ghoster",
        "str": "幽灵"
    },
    {
        "key": "2FEF48E44B224E75C8DE41B0CF76566A",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(2).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Lie bytes",
        "str": "谎言字节"
    },
    {
        "key": "D0C5B7DC4D0EDAB37A13E7A562E45ED6",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(3).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Chilling",
        "str": "颤栗者"
    },
    {
        "key": "F7F54678404761EAEB33B4B508EC97D1",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(4).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Hope",
        "str": "希望"
    },
    {
        "key": "45DB611B482D35EE05ADE98331F2EC20",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(5).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Haunting",
        "str": "阴魂不散"
    },
    {
        "key": "7B6CC6BA4D259033F5B946A472D9AB10",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(6).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Menacing",
        "str": "险恶者"
    },
    {
        "key": "0EBDA17F450C39EF22E70592BF2B4162",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(7).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Dreadful",
        "str": "恐惧者"
    },
    {
        "key": "BF221642449DB485C8FF39A6198C744B",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(8).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Sinister",
        "str": "阴险之徒"
    },
    {
        "key": "367FB31C49B8F835CBA9AAAF25735FD8",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.dolls_9_7FEB33284744601C3269539AAFDB6152(9).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Malicious",
        "str": "恶毒者"
    },
    {
        "key": "9A993AAB4A1C119EBBFB0FB94B27F8DE",
        "location": "/Game/data-tables/mapList.mapList.NewRow_1.name_18_82096130492E34A890A0138C4910D929",
        "id": "Daycare",
        "str": "托儿所"
    },
    {
        "key": "F4CB65EF48F9C01CD6AB509912890A36",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(0).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Ringmaster",
        "str": "马戏团团长"
    },
    {
        "key": "BCB8724F4D70F43E391D0FB8984329C6",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(1).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Diva",
        "str": "名媛"
    },
    {
        "key": "5AC930984185A24BC84F5EB1E734B80E",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(10).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Tubesocks",
        "str": "长筒袜"
    },
    {
        "key": "2CE416DB412AC40BEE092AAD7EC2F3F6",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(11).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Demon",
        "str": "恶魔"
    },
    {
        "key": "BAD425AD48B5C8A24B0663948D13EA94",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(12).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Planner",
        "str": "策划者"
    },
    {
        "key": "FACAB65943781B084CD803A77E0F7266",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(13).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Gurl",
        "str": "女孩"
    },
    {
        "key": "8482CDF142B6B8DA6AB0148C9B01430B",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(14).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Tactician",
        "str": "战术家"
    },
    {
        "key": "B56C952645AF8D8E34E6D7AC0CC9F750",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(15).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Loner",
        "str": "孤独者"
    },
    {
        "key": "9B0C4F0544EED1171CEEF59539778188",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(16).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Unlucky",
        "str": "倒霉蛋"
    },
    {
        "key": "E672CE194312B524EAE5218F788B97F4",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(17).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Stiff",
        "str": "呆板僵硬"
    },
    {
        "key": "B0A7B7E446CEF182A3AFDAB193FBBD80",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(18).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Grump",
        "str": "暴躁者"
    },
    {
        "key": "E238A88B4FBC7C4641B8E9A95353B1C0",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(19).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Indecent",
        "str": "下流者"
    },
    {
        "key": "A6E8E3B84059F699472F1C8DECDAF09C",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(2).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Star",
        "str": "明星"
    },
    {
        "key": "6599647B49FD6A15932490B717FE66F3",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(3).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Lover",
        "str": "爱人"
    },
    {
        "key": "B9B809EC40076554656E1A8C91157931",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(4).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Patriot",
        "str": "爱国者"
    },
    {
        "key": "01A2125C4F867B60C4E289ACD43FEBF9",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(5).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Mentor",
        "str": "导师"
    },
    {
        "key": "2C9826E7491425A6734CEB96CDC60470",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(6).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "Ms. Excellence",
        "str": "非凡女士"
    },
    {
        "key": "CAECC15549067E173F931888596593F1",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(7).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Sweet",
        "str": "甜蜜者"
    },
    {
        "key": "9CE4FD774DFBE4C84DC711A634666FA7",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(8).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Bot",
        "str": "机器人"
    },
    {
        "key": "9CDEB80D4E621DB2CC0B839965697925",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.dolls_9_7FEB33284744601C3269539AAFDB6152(9).dolls_9_7FEB33284744601C3269539AAFDB6152.alias_32_BCA92BEA4AAFD6928073EDABA20013F5",
        "id": "The Awkward",
        "str": "尴尬者"
    },
    {
        "key": "63063D214706E5FBBE2227836F1BE194",
        "location": "/Game/data-tables/mapList.mapList.NewRow_2.name_18_82096130492E34A890A0138C4910D929",
        "id": "Circus",
        "str": "马戏团"
    },
    {
        "key": "F2BC9C064F532236387F0ABC86BA8078",
        "location": "/Game/enums/actions.actions.DisplayNameMap(0 - Value).DisplayNameMap",
        "id": "Organization",
        "str": "整理"
    },
    {
        "key": "0078964D495E90CB9C263D86C289F115",
        "location": "/Game/enums/actions.actions.DisplayNameMap(1 - Value).DisplayNameMap",
        "id": "Tea with friends",
        "str": "与朋友喝茶"
    },
    {
        "key": "15973B6C49CE3AC92431D2B46BE8EEA4",
        "location": "/Game/enums/actions.actions.DisplayNameMap(10 - Value).DisplayNameMap",
        "id": "Fireworks",
        "str": "烟花"
    },
    {
        "key": "0DCDE313473E8A6D40F69ABCD969645B",
        "location": "/Game/enums/actions.actions.DisplayNameMap(11 - Value).DisplayNameMap",
        "id": "Swing Self",
        "str": "给自己荡秋千"
    },
    {
        "key": "7A7523B24363943AC1ECCE9A0B2634F5",
        "location": "/Game/enums/actions.actions.DisplayNameMap(12 - Value).DisplayNameMap",
        "id": "Balls",
        "str": "球"
    },
    {
        "key": "D8EB81F34ACA8EEE85F5E09D1DA0CB64",
        "location": "/Game/enums/actions.actions.DisplayNameMap(13 - Value).DisplayNameMap",
        "id": "Hide Self",
        "str": "隐藏自己"
    },
    {
        "key": "8785C0644A30D7680DD4668FB916A03A",
        "location": "/Game/enums/actions.actions.DisplayNameMap(14 - Value).DisplayNameMap",
        "id": "Hide Others",
        "str": "隐藏他人"
    },
    {
        "key": "9D7CA8BB4D12421F2093AB823A8EAC45",
        "location": "/Game/enums/actions.actions.DisplayNameMap(15 - Value).DisplayNameMap",
        "id": "Cannon",
        "str": "大炮"
    },
    {
        "key": "D8A8DBB34AAD96AC66B54A87365822C9",
        "location": "/Game/enums/actions.actions.DisplayNameMap(16 - Value).DisplayNameMap",
        "id": "Knives",
        "str": "刀子"
    },
    {
        "key": "0593C38042D49BC9D0958EB990324EFA",
        "location": "/Game/enums/actions.actions.DisplayNameMap(17 - Value).DisplayNameMap",
        "id": "Jail Self",
        "str": "监禁自己"
    },
    {
        "key": "61579A7B4A7A797C38911980E669DE96",
        "location": "/Game/enums/actions.actions.DisplayNameMap(18 - Value).DisplayNameMap",
        "id": "Jail Others",
        "str": "监禁他人"
    },
    {
        "key": "FF934248449771227CA5E6BAC5FBE7AF",
        "location": "/Game/enums/actions.actions.DisplayNameMap(19 - Value).DisplayNameMap",
        "id": "Carousel with friends",
        "str": "和朋友一起玩旋转木马"
    },
    {
        "key": "FD50C07D436B39761D90A28961DE809F",
        "location": "/Game/enums/actions.actions.DisplayNameMap(2 - Value).DisplayNameMap",
        "id": "Open Boxes",
        "str": "打开盒子"
    },
    {
        "key": "9432768D4222957489019C86BD251CA3",
        "location": "/Game/enums/actions.actions.DisplayNameMap(20 - Value).DisplayNameMap",
        "id": "Carousel with enemies",
        "str": "和敌人共乘旋转木马"
    },
    {
        "key": "420F5BCA40413C0383C7AA9583EEC5C0",
        "location": "/Game/enums/actions.actions.DisplayNameMap(21 - Value).DisplayNameMap",
        "id": "Mislead",
        "str": "误导"
    },
    {
        "key": "E2D484854A1C9DB14835A6AD457B6207",
        "location": "/Game/enums/actions.actions.DisplayNameMap(22 - Value).DisplayNameMap",
        "id": "Balloons",
        "str": "气球"
    },
    {
        "key": "5E1C23674444BB331554B39281C7EA20",
        "location": "/Game/enums/actions.actions.DisplayNameMap(23 - Value).DisplayNameMap",
        "id": "Swing Others",
        "str": "给他人荡秋千"
    },
    {
        "key": "87FBE09C4BC5718F45CC6F8E716201AE",
        "location": "/Game/enums/actions.actions.DisplayNameMap(25 - Value).DisplayNameMap",
        "id": "Tunnels",
        "str": "隧道"
    },
    {
        "key": "49EE0D0C418246C21899368F2615D0DE",
        "location": "/Game/enums/actions.actions.DisplayNameMap(26 - Value).DisplayNameMap",
        "id": "Trains",
        "str": "列车"
    },
    {
        "key": "8E2C567C4DD801B3FD73AFB9C374D6B4",
        "location": "/Game/enums/actions.actions.DisplayNameMap(27 - Value).DisplayNameMap",
        "id": "Cars",
        "str": "汽车"
    },
    {
        "key": "A7C36D644573AD30716F4283EBBE53D2",
        "location": "/Game/enums/actions.actions.DisplayNameMap(28 - Value).DisplayNameMap",
        "id": "Swings",
        "str": "秋千"
    },
    {
        "key": "D627A24C4957566043D3F0ADF0F70B9E",
        "location": "/Game/enums/actions.actions.DisplayNameMap(29 - Value).DisplayNameMap",
        "id": "Playgrounds",
        "str": "游乐场"
    },
    {
        "key": "AA8520BE4876828E544192BFA8F98EE7",
        "location": "/Game/enums/actions.actions.DisplayNameMap(3 - Value).DisplayNameMap",
        "id": "Move Self",
        "str": "移动自己"
    },
    {
        "key": "455698874FAE4AEB808E15AA36758F84",
        "location": "/Game/enums/actions.actions.DisplayNameMap(30 - Value).DisplayNameMap",
        "id": "Houses",
        "str": "房屋"
    },
    {
        "key": "06E2AE2A4553C0E1B8BD4FAEBFD8FA1E",
        "location": "/Game/enums/actions.actions.DisplayNameMap(31 - Value).DisplayNameMap",
        "id": "Seesaws",
        "str": "跷跷板"
    },
    {
        "key": "0285258748FED1BC6E542F80A9097CE9",
        "location": "/Game/enums/actions.actions.DisplayNameMap(32 - Value).DisplayNameMap",
        "id": "Tea",
        "str": "茶"
    },
    {
        "key": "8DDD1FD04F1754E2F728208EBEF897FC",
        "location": "/Game/enums/actions.actions.DisplayNameMap(33 - Value).DisplayNameMap",
        "id": "Camp with friends",
        "str": "和朋友一起露营"
    },
    {
        "key": "591DF6954C2D384CBCCD838B12BD62CA",
        "location": "/Game/enums/actions.actions.DisplayNameMap(34 - Value).DisplayNameMap",
        "id": "Camp with enemies",
        "str": "和敌人露营"
    },
    {
        "key": "1F51194C47FBB1A80E8868BDC35CC619",
        "location": "/Game/enums/actions.actions.DisplayNameMap(35 - Value).DisplayNameMap",
        "id": "Sandbox",
        "str": "沙盒"
    },
    {
        "key": "0AB50D9E41B57073B2D0EFAA0AD0A9F4",
        "location": "/Game/enums/actions.actions.DisplayNameMap(36 - Value).DisplayNameMap",
        "id": "Bouncy castles",
        "str": "充气城堡"
    },
    {
        "key": "F234C7F94700549E267ADA87AC492287",
        "location": "/Game/enums/actions.actions.DisplayNameMap(37 - Value).DisplayNameMap",
        "id": "Ouija",
        "str": "通灵板"
    },
    {
        "key": "E0DDAB024B5C9B1C6FD0A8BAB68ED718",
        "location": "/Game/enums/actions.actions.DisplayNameMap(38 - Value).DisplayNameMap",
        "id": "Radio",
        "str": "收音机"
    },
    {
        "key": "5F6925D947AE92C9AC54DEA293653C03",
        "location": "/Game/enums/actions.actions.DisplayNameMap(39 - Value).DisplayNameMap",
        "id": "Rocking",
        "str": "摇摆"
    },
    {
        "key": "FF277FA44BB3DB99FB3F4393911D9B6D",
        "location": "/Game/enums/actions.actions.DisplayNameMap(4 - Value).DisplayNameMap",
        "id": "Music",
        "str": "音乐"
    },
    {
        "key": "0D7E18B14371021AF672E392EB48BEA1",
        "location": "/Game/enums/actions.actions.DisplayNameMap(40 - Value).DisplayNameMap",
        "id": "Fireplace",
        "str": "火炉"
    },
    {
        "key": "1AEBF7804411904275F320B5DB9BB455",
        "location": "/Game/enums/actions.actions.DisplayNameMap(41 - Value).DisplayNameMap",
        "id": "Candles",
        "str": "蜡烛"
    },
    {
        "key": "E321649D4AF8734E473FEDB0404E1B2E",
        "location": "/Game/enums/actions.actions.DisplayNameMap(42 - Value).DisplayNameMap",
        "id": "Move frames",
        "str": "移动框架"
    },
    {
        "key": "9C3A179A4105F07AF02DAF9565B1CB02",
        "location": "/Game/enums/actions.actions.DisplayNameMap(43 - Value).DisplayNameMap",
        "id": "Clock",
        "str": "钟"
    },
    {
        "key": "60929102471030286F6536A92C1EC5C8",
        "location": "/Game/enums/actions.actions.DisplayNameMap(44 - Value).DisplayNameMap",
        "id": "Book",
        "str": "书"
    },
    {
        "key": "24FA60AC4C99AD7D35B45C8F7E6C1279",
        "location": "/Game/enums/actions.actions.DisplayNameMap(45 - Value).DisplayNameMap",
        "id": "Chess",
        "str": "棋"
    },
    {
        "key": "3753BCEE46C70EDFBC5171BE7A651CC0",
        "location": "/Game/enums/actions.actions.DisplayNameMap(46 - Value).DisplayNameMap",
        "id": "Curtains",
        "str": "窗帘"
    },
    {
        "key": "C9DD78084F647953E55A8196CE872921",
        "location": "/Game/enums/actions.actions.DisplayNameMap(47 - Value).DisplayNameMap",
        "id": "Gramophone",
        "str": "留声机"
    },
    {
        "key": "18D93C0E4280B2D8733C3B9098852779",
        "location": "/Game/enums/actions.actions.DisplayNameMap(48 - Value).DisplayNameMap",
        "id": "Magic trick",
        "str": "魔术把戏"
    },
    {
        "key": "733845AE414C327753B93094F83BF162",
        "location": "/Game/enums/actions.actions.DisplayNameMap(49 - Value).DisplayNameMap",
        "id": "Balloon",
        "str": "气球"
    },
    {
        "key": "19750FC24F8A0B2279246B801F612D34",
        "location": "/Game/enums/actions.actions.DisplayNameMap(5 - Value).DisplayNameMap",
        "id": "Blood",
        "str": "血液"
    },
    {
        "key": "54CD20444D2BAF2E0A417794EE4D21F9",
        "location": "/Game/enums/actions.actions.DisplayNameMap(50 - Value).DisplayNameMap",
        "id": "Carousel",
        "str": "旋转木马"
    },
    {
        "key": "09B43DB3406AF2166732B48B577F4E20",
        "location": "/Game/enums/actions.actions.DisplayNameMap(51 - Value).DisplayNameMap",
        "id": "Crystal Ball",
        "str": "水晶球"
    },
    {
        "key": "80178DF845B4F8C8D4B5DABBDC29F983",
        "location": "/Game/enums/actions.actions.DisplayNameMap(52 - Value).DisplayNameMap",
        "id": "Firecrackers",
        "str": "鞭炮"
    },
    {
        "key": "ECA7B6BD4DC5448D5BC007A0A56EFE97",
        "location": "/Game/enums/actions.actions.DisplayNameMap(53 - Value).DisplayNameMap",
        "id": "Dollhouse",
        "str": "人偶屋"
    },
    {
        "key": "0177D62B4B9CBCAFEE5A968A55483181",
        "location": "/Game/enums/actions.actions.DisplayNameMap(54 - Value).DisplayNameMap",
        "id": "Blackboard",
        "str": "黑板"
    },
    {
        "key": "2F564E2B4FB1B7CB88A558A4D73A0B22",
        "location": "/Game/enums/actions.actions.DisplayNameMap(55 - Value).DisplayNameMap",
        "id": "Canvas",
        "str": "画布"
    },
    {
        "key": "3F370B9043F249CD818C5C9E9D7218CD",
        "location": "/Game/enums/actions.actions.DisplayNameMap(56 - Value).DisplayNameMap",
        "id": "Words",
        "str": "词语"
    },
    {
        "key": "62E75F3344325D96C309BFB52949F95A",
        "location": "/Game/enums/actions.actions.DisplayNameMap(57 - Value).DisplayNameMap",
        "id": "Stuffed Animals",
        "str": "毛绒动物"
    },
    {
        "key": "66D14BB44970D4F17A0184929B3FE083",
        "location": "/Game/enums/actions.actions.DisplayNameMap(58 - Value).DisplayNameMap",
        "id": "Animated Stuffed animal",
        "str": "栩栩如生的毛绒动物"
    },
    {
        "key": "5DFAFA884E37BBC8C2C6959F8EF168A7",
        "location": "/Game/enums/actions.actions.DisplayNameMap(6 - Value).DisplayNameMap",
        "id": "Move Others",
        "str": "移动他人"
    },
    {
        "key": "F343C45143C6D6F84AC69C85B3A85B2F",
        "location": "/Game/enums/actions.actions.DisplayNameMap(7 - Value).DisplayNameMap",
        "id": "Tv",
        "str": "电视"
    },
    {
        "key": "21FFAF7243B370CCA19D849D0F92E7A4",
        "location": "/Game/enums/actions.actions.DisplayNameMap(8 - Value).DisplayNameMap",
        "id": "Tea with enemies",
        "str": "与敌人喝茶"
    },
    {
        "key": "D2BAA1EC4F84BE5AA48BBC92F65D2FA8",
        "location": "/Game/enums/actions.actions.DisplayNameMap(9 - Value).DisplayNameMap",
        "id": "Light",
        "str": "光明"
    },
    {
        "key": "9639E2074CB67CEC4666A4AF3A4FC559",
        "location": "/Game/enums/costs.costs.DisplayNameMap(0 - Value).DisplayNameMap",
        "id": "Very Cheap",
        "str": "非常便宜"
    },
    {
        "key": "73D31FDE441CC82C2ADFE9B2A3A7A293",
        "location": "/Game/enums/costs.costs.DisplayNameMap(1 - Value).DisplayNameMap",
        "id": "Cheap",
        "str": "便宜"
    },
    {
        "key": "AD00137D457BF38AE614FC9EE94D1425",
        "location": "/Game/enums/costs.costs.DisplayNameMap(2 - Value).DisplayNameMap",
        "id": "Expensive",
        "str": "昂贵"
    },
    {
        "key": "77597C9F40C4017557D06AA387E711B0",
        "location": "/Game/enums/costs.costs.DisplayNameMap(3 - Value).DisplayNameMap",
        "id": "Very Expensive",
        "str": "非常昂贵"
    },
    {
        "key": "3AD41353481D8F51B3C3699171A4CED1",
        "location": "/Game/enums/gamemodes.gamemodes.DisplayNameMap(0 - Value).DisplayNameMap",
        "id": "Classic",
        "str": "经典"
    },
    {
        "key": "249DE7534CD1567EA10B7F971D9C0BF1",
        "location": "/Game/enums/gamemodes.gamemodes.DisplayNameMap(1 - Value).DisplayNameMap",
        "id": "Career",
        "str": "生涯"
    },
    {
        "key": "D107C133465E950C0D0B42BAAA8BDF09",
        "location": "/Game/enums/probability.probability.DisplayNameMap(0 - Value).DisplayNameMap",
        "id": "Very Low",
        "str": "非常低"
    },
    {
        "key": "626EC2FD45418B0B7A54ADBCAEADA3A0",
        "location": "/Game/enums/probability.probability.DisplayNameMap(1 - Value).DisplayNameMap",
        "id": "Low",
        "str": "低"
    },
    {
        "key": "E60C061E42EEE57516BA1B83B2A64C7E",
        "location": "/Game/enums/probability.probability.DisplayNameMap(2 - Value).DisplayNameMap",
        "id": "Normal",
        "str": "正常"
    },
    {
        "key": "9404939C41414F86FA16B8B0C5B74E9D",
        "location": "/Game/enums/probability.probability.DisplayNameMap(3 - Value).DisplayNameMap",
        "id": "High",
        "str": "高"
    },
    {
        "key": "A87033014B68DAC4F5312BB46BEB28AF",
        "location": "/Game/enums/probability.probability.DisplayNameMap(4 - Value).DisplayNameMap",
        "id": "Very High",
        "str": "非常高"
    },
    {
        "key": "7B965EEB441BE13C098439BB745C2F25",
        "location": "/Game/functions/craftFunctions.craftFunctions_C:craftItem [Script Bytecode]",
        "id": "You're short of ingredients to make this",
        "str": "原料不足"
    },
    {
        "key": "DCBD9A3F418F5F127DABDD996614C5B0",
        "location": "/Game/functions/craftFunctions.craftFunctions_C:craftItem [Script Bytecode]",
        "id": "You can't make this, it doesn't fit any recipe",
        "str": "没有配方，不能这么做"
    },
    {
        "key": "003123EB4E325C82B8C7E192D319CF55",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Creepy",
        "str": "阴森之物"
    },
    {
        "key": "00BD15F04FF6DC2CA86C84A59FC088F8",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Darkling",
        "str": "黑暗之子"
    },
    {
        "key": "02866DAB46A2CC7C42F7DBB8F29C8616",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Echo",
        "str": "回响"
    },
    {
        "key": "042571704BB706480B0FDD806AE57897",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Morbid",
        "str": "病态之人"
    },
    {
        "key": "0600D2134F5944FF6E3102866C348F23",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Enigma",
        "str": "谜团"
    },
    {
        "key": "0654BE994FA7D802E7973F901694CDD2",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Gasp",
        "str": "喘息者"
    },
    {
        "key": "14427A08459289522231F5B3835D2594",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Seer",
        "str": "先知"
    },
    {
        "key": "14BCFAF24A161704AE43A69E6EB83452",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Enigmatic",
        "str": "神秘莫测"
    },
    {
        "key": "156635BF49E1B3C111682A92C49899D7",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Murmur",
        "str": "低声呢喃"
    },
    {
        "key": "158063504E027E4AF2D595A2CE468770",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Phantasm",
        "str": "幻象"
    },
    {
        "key": "176C171C443DFFFD974A32BB32959193",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Obscura",
        "str": "The Obscura"
    },
    {
        "key": "1AF5392F481406F80C566FB0740D5D3D",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Shady",
        "str": "乘凉者"
    },
    {
        "key": "1B0B52154F304355C2F9DFBA6325F8BB",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Ghast",
        "str": "恶魂"
    },
    {
        "key": "1E5D3DCD4F80B7273B0E9882F51B5270",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Howl",
        "str": "嚎叫者"
    },
    {
        "key": "1EB2085245408A6B326DDF9AA9214E0A",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Glint",
        "str": "熠熠生辉"
    },
    {
        "key": "1F7140B7494998788B096E85BEA451B4",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Whispers",
        "str": "低语者"
    },
    {
        "key": "20522EAA4B93A60CC0A3D19D27066678",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Grimace",
        "str": "怪相"
    },
    {
        "key": "212CE2FB4F7C398C125B5C8C8915BB95",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Cackle",
        "str": "窃笑者"
    },
    {
        "key": "22BFA453461BA510FCB07C9231AA8F6C",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Silhouette",
        "str": "幽影"
    },
    {
        "key": "2777F4DA4703D3AC622572A49EB6A138",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Stalker",
        "str": "跟踪狂"
    },
    {
        "key": "2A0B22FD46CEA7F67C28AAA09A2DBB92",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Grin",
        "str": "嬉笑者"
    },
    {
        "key": "2A63AABC4C73EF73FB658F9F39376B6B",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Whisperwind",
        "str": "风言风语"
    },
    {
        "key": "2AD4B4974FD1B23943AF91A6E5D447E8",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Ghoul",
        "str": "食尸鬼"
    },
    {
        "key": "32053A4646B4374EA4DBD3BC64CB936C",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Spirit",
        "str": "魂魄"
    },
    {
        "key": "3353AF434D2A72322E22DB981CE05EAA",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Twilight",
        "str": "暮光"
    },
    {
        "key": "37B6780C449B80C581D1478BD1CB6A4A",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Veiled",
        "str": "蒙面者"
    },
    {
        "key": "37E26C5949E57421072DF6AFA351D3FD",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Moan",
        "str": "哀悼者"
    },
    {
        "key": "3AEB5BB747D3C431D54680AEFDB7D404",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Silent",
        "str": "无言者"
    },
    {
        "key": "3DBA3FB1494E536000C8F997B62EA3C1",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Mournful",
        "str": "悲痛者"
    },
    {
        "key": "3F845D654ED6C33DC540F4BC414F7994",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Ebon",
        "str": "乌黑"
    },
    {
        "key": "40CA9F4E4306D720405AF5BA3B5DFEE5",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Bone",
        "str": "骨头"
    },
    {
        "key": "446183A644D53693C3BF6E947D141A43",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Hauntling",
        "str": "萦绕幼魂"
    },
    {
        "key": "4798B1E54C73C4932878B1A4630376ED",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Whispering",
        "str": "耳语者"
    },
    {
        "key": "4A335723409BCF0895C868A63CFBA1DB",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Nightfall",
        "str": "夜幕降临"
    },
    {
        "key": "4E8841A242B9882466D2B88B44A43CC6",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Veil",
        "str": "恐怖面纱"
    },
    {
        "key": "51205E6D45797595CB5B90B13FC6BF54",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Uncanny",
        "str": "怪诞"
    },
    {
        "key": "5E6EA38E411F23949A2E5D828B409DDF",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Umbral",
        "str": "深黯幽影"
    },
    {
        "key": "60E9BFBD4D9BFF7D7952CA9115FFAA10",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Shadow",
        "str": "暗影"
    },
    {
        "key": "63E5DF9A4DC914E24DCDA8B02FC0A001",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Wraith",
        "str": "魅影"
    },
    {
        "key": "6583C6EE4A8FDE826EE589BCC51DF291",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Riddle",
        "str": "谜"
    },
    {
        "key": "67B677CF4F7F5F5336B2A5A78A824E51",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Twisted",
        "str": "扭曲之魂"
    },
    {
        "key": "6A0CBE8D405C375978D82F96CC069A3F",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Lament",
        "str": "挽歌"
    },
    {
        "key": "6AE175A04F5F286311D601922779AEE5",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Dire",
        "str": "极端恐惧"
    },
    {
        "key": "6CBA009E433B551763CA96B31AF8596B",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Midnight",
        "str": "午夜"
    },
    {
        "key": "6FA03C18423CD639AF92D892EA7A83E3",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Hallowed",
        "str": "神圣之魂"
    },
    {
        "key": "706E8DB14C70C3DE52A34BA3930A04B1",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Ghostly",
        "str": "恐怖幽灵"
    },
    {
        "key": "70EEFB58402BFE6D2679A59821BAFB3F",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Glare",
        "str": "怒目而视"
    },
    {
        "key": "77FB63CC4DE70AC03435D38AAA0FDBE5",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Echosong",
        "str": "回响之歌"
    },
    {
        "key": "7969D08443D6AB359F1A76AFDC805E0F",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Murmurer",
        "str": "低声埋怨"
    },
    {
        "key": "7B05AC704AB111B1076AAFBB3A5C66E5",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Hush",
        "str": "缄默"
    },
    {
        "key": "7B5021F44C29F7525F8D74A05BB722CC",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Lurk",
        "str": "潜伏者"
    },
    {
        "key": "80B42CA54800387D3889EDBDBB0E290E",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Spectral",
        "str": "徘徊幽灵"
    },
    {
        "key": "81F23CF240C4A672B1AC05A6895DF582",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Pale",
        "str": "苍白幽魂"
    },
    {
        "key": "84743C7D412BD85DD93CC092751C89E7",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Murk",
        "str": "阴暗之魂"
    },
    {
        "key": "861398CE4A33BD25FB23BE95A3EC66C6",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Shroud",
        "str": "裹尸"
    },
    {
        "key": "87B1E3F54AA52AE964B7BDAF99914042",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Whispering Breeze",
        "str": "低语微风"
    },
    {
        "key": "8B7BEED04A64FDBA38C388891DF8F479",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Hollow",
        "str": "空洞灵魂"
    },
    {
        "key": "91F60C4F49676E09BAD1F2BC4320EEC5",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Lurking",
        "str": "潜伏幽魂"
    },
    {
        "key": "986FE18441EBE7BFF160D38E25BEF05E",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Phantom Breeze",
        "str": "幻影微风"
    },
    {
        "key": "99BCFC05431F42E7A29B539406D75C0B",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Screech",
        "str": "尖叫者"
    },
    {
        "key": "9AA78476434241C7EE4929A6B33EE813",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Shifting Shadow",
        "str": "潜影"
    },
    {
        "key": "9B283B084E6D48B26573D0A0CD07A07E",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Menace",
        "str": "恐吓者"
    },
    {
        "key": "9CD486614893E83F1AC2D3AA683226F0",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Gloaming",
        "str": "薄暝"
    },
    {
        "key": "9CF4BD434FB00A0E98BE348036F66B3F",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Shade",
        "str": "阴影"
    },
    {
        "key": "9DF350484FB5F72D90CC1D9B5AE80C31",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Obscure",
        "str": "潜匿幽魂"
    },
    {
        "key": "A25E20BF4E349221F9E980AF8435D2FD",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Wicked",
        "str": "邪恶的"
    },
    {
        "key": "A43C17F940FC87439BFB25A6FAC1FC63",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Abyss",
        "str": "深渊"
    },
    {
        "key": "A5B8E42E465FF909827ACBAEAFD09986",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Murky",
        "str": "昏暗"
    },
    {
        "key": "A7F922E44402D03BA8B2DCA5ED3EDB54",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Dark",
        "str": "暗魂"
    },
    {
        "key": "AC61D57E4AA1B3D2ED8E35BD60ADA0B2",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Phantom",
        "str": "幻影"
    },
    {
        "key": "B16BD65740B9A94C398B138BCEEA9678",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Flicker",
        "str": "闪烁幽魂"
    },
    {
        "key": "B190DC8C4B38B72150B586B737F83910",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Shadowed Whisper",
        "str": "暗影低语者"
    },
    {
        "key": "B5E4018D483BA5D32A8B06AF41CF03FC",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Whispered",
        "str": "窃窃私语"
    },
    {
        "key": "B708786A4C2E1065BA0B60A6CAA2FAC9",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Scowl",
        "str": "愁容满面"
    },
    {
        "key": "C0D16AB6449869CB93491684F3F5D44C",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Eerie",
        "str": "古灵精怪"
    },
    {
        "key": "C6298185486DF0A5C64519AD8B518D4C",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Shadowling",
        "str": "小阴影"
    },
    {
        "key": "D0B7B2AA420A8149B92FA5A08DC01FFE",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Cryptic",
        "str": "咯咯笑"
    },
    {
        "key": "D20B4CF442582F2B1E945BA082899AD3",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Echoing",
        "str": "余音缭绕"
    },
    {
        "key": "D400DBFE4DD655EF2BD09C888BD1C988",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Gloom",
        "str": "愁容满面"
    },
    {
        "key": "D4EF88484AEDF13F230DCB808F4F9D2E",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Creep",
        "str": "爬行者"
    },
    {
        "key": "D4F0469A4306A9237BF0858B3B2541EF",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Shudder",
        "str": "颤栗之魂"
    },
    {
        "key": "D54AD32246C068D95A3BF5911FDC1807",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Lurker",
        "str": "潜伏之魂"
    },
    {
        "key": "D9E1915C4338A30647EBA6BB3EEB4D2C",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Wisp",
        "str": "鬼火"
    },
    {
        "key": "E83CDA864E3FD82D333504A00D6E24D4",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Omen",
        "str": "噩兆"
    },
    {
        "key": "E85D23F14D7C46E45DA387A747928F07",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Whisper",
        "str": "轻声低语"
    },
    {
        "key": "E9F12A4243E6AE323515DE8E9AA0162B",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Specter",
        "str": "鬼影"
    },
    {
        "key": "EEEE750B4277BD6C5835BD901B924179",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Dusk",
        "str": "黄昏"
    },
    {
        "key": "F28B41324F202FA2E02A0DB59579D291",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Wretched",
        "str": "卑劣之魂"
    },
    {
        "key": "F5698F3144BF19F3BDD3D58AA167101F",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Whisperer",
        "str": "告密者"
    },
    {
        "key": "F58DB0B646AB1214A995FD9D228448FD",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Dread",
        "str": "恐惧之魂"
    },
    {
        "key": "F68B1482480395C0AD66298BBA07A363",
        "location": "/Game/functions/dollFunctions.dollFunctions_C:getDynamicAlias [Script Bytecode]",
        "id": "The Dive",
        "str": "深潜者"
    },
    {
        "key": "2355C0C345515E1D7874728AF33121DC",
        "location": "/Game/functions/dynamicWorldFunctions.dynamicWorldFunctions_C:setPhoto [Script Bytecode]",
        "id": "Multiple targets have appeared in the photo; try to get closer and focus only on the target you want.",
        "str": "照片中出现了多个目标；试着靠近目标，只关注可疑的目标。"
    },
    {
        "key": "26C6D54D41B3F28C6F8033ADBE044F08",
        "location": "/Game/functions/dynamicWorldFunctions.dynamicWorldFunctions_C:setPhoto [Script Bytecode]",
        "id": "The target has not been detected, try to focus more and stay at a specific distance where the entire target can be seen",
        "str": "还未发现目标，试着集中更多的注意力，保持在特定的距离，看见整个目标"
    },
    {
        "key": "93B997314AF656293CD3E6A5CB67FA1B",
        "location": "/Game/functions/messageFunctions.messageFunctions_C:getErrorText [Script Bytecode]",
        "id": "Error",
        "str": "错误"
    },
    {
        "key": "5154AFE04B70A5D272398C80C9353C25",
        "location": "/Game/functions/messageFunctions.messageFunctions_C:getNeedText [Script Bytecode]",
        "id": "You need {name}",
        "str": "你需要{name}"
    },
    {
        "key": "AD84DE7C441EA1A5D211A5AA27D02D95",
        "location": "/Game/functions/messageFunctions.messageFunctions_C:getPressText [Script Bytecode]",
        "id": "Press {key} to {interact}",
        "str": "按{key}{interact}"
    },
    {
        "key": "C158D252413F592114D488BB11C8055E",
        "location": "/Game/functions/messageFunctions.messageFunctions_C:getRequireEnergyText [Script Bytecode]",
        "id": "Require Energy",
        "str": "需要能量"
    },
    {
        "key": "5BC1D7604A1C399EA23C8A967E53983E",
        "location": "/Game/functions/netFunctions.netFunctions_C:kickPlayer [Script Bytecode]",
        "id": "Reason: {reason}",
        "str": "原因：{reason}"
    },
    {
        "key": "6F579E3E4959DC13FA5EA69C981AA6CC",
        "location": "/Game/functions/netFunctions.netFunctions_C:kickPlayer [Script Bytecode]",
        "id": "You are kicked",
        "str": "你被踢了"
    },
    {
        "key": "0D80C37D480FE34BE1CA64878A0D1B7D",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now the radio",
        "str": "现在是收音机"
    },
    {
        "key": "158F6D234AF05145B4905F942285D978",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now the Ouija board on the table",
        "str": "现在是桌子上的通灵板"
    },
    {
        "key": "15A53736473F7B173C354496D5FF2638",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Very well, you've used all the items on the doll. This time, the things that Lil didn't like were: {items}",
        "str": "很好，你已经用完了所有的物品。这次，Lil 不喜欢的东西是：{items}"
    },
    {
        "key": "16B725324055549E77710CACDD3325FF",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now move the picture closest to the living room door.",
        "str": "现在移动最靠近客厅门的图片。"
    },
    {
        "key": "20A3D5CB40F0360E6C38BABBA2706D90",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "These elixirs reveal relationships between the dolls. They all have relationships, and this is the only way to discover them. Let's start by testing Lil's love relationship with the other dolls. Simply apply the elixir to Lil and observe the effects. All those who respond to Lil's effects are the ones Lil loves.",
        "str": "这些药剂揭示了人偶之间的关系。他们之间都有关系，这是发现关系的唯一方法。让我们从测试 Lil 与其他人偶的爱情关系开始。只需将药剂涂在 Lil 身上并观察效果。所有对 Lil 的效果做出反应的人都是 Lil 爱的人。"
    },
    {
        "key": "230AAACD49536A9594B0CBB4CA653C76",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now the tv",
        "str": "现在是电视"
    },
    {
        "key": "2A85705945159E3B98E06BB40B5CA820",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now the book",
        "str": "现在是书"
    },
    {
        "key": "2EA5F3054E8CD308869D52B17D3DE346",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "I've also given you a bag. This bag will help you capture the impostor. Once you catch it, the game will end (if you do so within the time limit). I'll leave you to finish this part of the tutorial. Whether you win or lose, you're done. Good luck!",
        "str": "现在我也给了你一个袋子。这个袋子将帮助你抓住冒名顶替者。一旦你抓住它，游戏就会结束（如果你在时间限制内完成）。我会让你自己完成这部分教程。无论你赢还是输，都算完成了。祝你好运！"
    },
    {
        "key": "336B557643CBA00B5138539520DCBB04",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Let's start with the chair outside.",
        "str": "让我们从外面的椅子开始。"
    },
    {
        "key": "3C68D11C43CC3EA53113F48A08FDF3AD",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now the wall clock",
        "str": "现在是挂钟"
    },
    {
        "key": "3E8440AC413D7725FE75218355F55B7B",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Great! I've given you the dark paper. Use it on Lil and write down everything you see. Remember that these are the things she doesn't like, so anything she extinguishes will indicate her dislikes.",
        "str": "太棒了！我给了你暗纸。用它在Lil身上，写下你看到的一切。记住，这些是她不喜欢的东西，所以她熄灭的任何东西都会表明她的不喜欢。"
    },
    {
        "key": "3FAB1E074AD22684E3F78785D85679FA",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now the nearest candle.",
        "str": "现在最近的蜡烛。"
    },
    {
        "key": "4715107549F6817B3883F99871D34055",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Leave the room now and check everything that’s activated. Write down everything you see in the doll's likes!",
        "str": "现在离开房间，检查所有已激活的内容。写下你在娃娃喜欢的东西中看到的一切！"
    },
    {
        "key": "4E0811124112D56E159C9290EA5B7FEF",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "I've given you a notebook where you can see all the details about the dolls and complete information. Take a look, and when you're ready, come back to the dolls.",
        "str": "我给了你一个笔记本，你可以在里面看到所有关于人偶的细节和完整信息。看一看，准备好后，回到人偶那里。"
    },
    {
        "key": "520103D94DEC134A00BBF6ABA778AA0F",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Well done, you've completed the elixirs. I'm not checking your notes, but I'm sure they're great!",
        "str": "做得好，你完成了药剂。我不会检查你的笔记，但我相信它们很棒！"
    },
    {
        "key": "58232C9B4DC256E252E24DA64FBF074E",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Before giving you the dark paper, let's activate and turn everything on. This way, it will be easier for us to see what the doll dislikes when we stop it.",
        "str": "在给你暗纸之前，让我们激活并打开所有东西。这样，当我们停下来时，我们更容易看到人偶不喜欢什么。"
    },
    {
        "key": "6342E3B646E7260C285568B47D8CEA82",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "When you start a case, you'll be able to equip yourself with the items you've bought or crafted. To do this, access the chest behind you. Right now, you'll see it empty, but don't worry, I'll be giving you items so you can get familiar with them!",
        "str": "当你开始一个案件时，你将能够装备你购买或制作的物品。要做到这一点，访问你身后的箱子。现在你会看到它是空的，但不用担心，我会给你物品，这样你就可以熟悉它们！"
    },
    {
        "key": "67CC4CE342660CF969980E8F00DC0788",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Very well, now the remaining candle.",
        "str": "很好，现在剩下蜡烛。"
    },
    {
        "key": "69E108EB47016FEFAFF123ACC82F5F25",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Perfect, now let's investigate one of the dolls. I've given you a camera; let's take a picture of Lil.",
        "str": "很好，现在让我们调查其中一个人偶。我给了你一个相机；让我们给Lil拍张照片。"
    },
    {
        "key": "7FA2116B4EBBB3EAA099768D250E6ADF",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "The effects have worn off. If you didn’t manage to write it down, Lil’s loves are: {people}",
        "str": "效果已经消失。如果你没记下来，Lil的爱好是：{people}"
    },
    {
        "key": "84EDA79E484E05EF27934DBA2708C4CD",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Electricity On! Head back to the house.",
        "str": "电力开启！回到屋子里。"
    },
    {
        "key": "96567B804DFD2EED05E51E9BD49F3870",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now we'll use the Soul Book item. This item will officially start the game. Its use is simple: when you use this book, whoever the impostor is (in this case, Lil) will begin to take actions. At this point, you'll need to compare the information with what happens, just like in the original mode.",
        "str": "现在我们将使用灵魂之书物品. 该物品将正式开始游戏. 它的使用很简单：当你使用这本书时，无论冒名顶替者是谁（在本例中为Lil）都会开始采取行动. 此时，你将"
    },
    {
        "key": "9B8ABC7E40B142594630D5A2C61ECC9A",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Elixir applied! Check the dolls around to determine whom Lil loves.",
        "str": "elixir已使用！检查周围的娃娃，确定Lil爱谁。"
    },
    {
        "key": "9DC1A4244661454B29521687D81248D8",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Let's cross off Lil since that's the doll we're going to investigate.",
        "str": "让我们划掉Lil，因为这是我们要调查的娃娃。"
    },
    {
        "key": "9EE1B4694260CACEFC64E49A837B3C23",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "To develop the photo, you need to move it. Once it's 100% developed, simply press the use button, and it will be saved.",
        "str": "要处理照片，您需要移动它。一旦它100%处理完毕，只需按下使用按钮，它就会被保存。"
    },
    {
        "key": "A5E0C01D496E526AB4FF1FBD31F0B545",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Time's up! If you didn't manage to write everything down, here's what was observed: {actions}",
        "str": "时间到！如果你没能记下所有的东西，这是观察到的内容：{actions}"
    },
    {
        "key": "B06763C640AF25AF95E932927F9E6D59",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Alright, I'll give you the rest of the elixirs to apply to Lil. Amicitia represents friendship, Inimicitia represents enmity, and Odium represents hatred. Each of these, with its specific color, will reveal Lil's relationships. Write down everything you observe, and remember that in some cases, Lil might not have any relationship with a specific role.",
        "str": "好的，我会给你剩下的elixirs，让你用在Lil身上。Amicitia代表友谊，Inimicitia代表敌意，Odium代表仇恨。每种都有特定的颜色，会揭示Lil的关系。记下你观察到的一切，记住，在某些情况下，Lil可能与特定角色没有任何关系。"
    },
    {
        "key": "B324B28C4466BEA72263AD9E23F8755D",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "I've given you a lighter. Use it for the remaining actions.",
        "str": "我给了你一个打火机。用它来完成剩下的动作。"
    },
    {
        "key": "BB41F17C4CF0DBCE79EC7581B01FF65F",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now you should write them down in the notes of the notepad. Additionally, if you're using a keyboard and mouse, and you click outside the notes, the doll's page will remain open.",
        "str": "现在你应该把它们写在记事本的笔记中。此外，如果你使用键盘和鼠标，并且在笔记外单击，人偶页面将保持打开状态。"
    },
    {
        "key": "BC337075457AE8659374D6BF82D6E51D",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "We've captured Lil. You can continue photographing the rest of the dolls. When you're ready, head to the caravan to view the photos.",
        "str": "我们已经抓住了Lil。你可以继续拍摄剩下的人偶。准备好后，前往房车查看照片。"
    },
    {
        "key": "BF5B4BBA458403674A2BD89F8DC120A5",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Now table chess",
        "str": "现在是棋盘游戏"
    },
    {
        "key": "C380BEFB4806636D471F28BC3C7CE999",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Soul Papers",
        "str": "灵魂纸"
    },
    {
        "key": "C6EE1D854906A3E6B08CFF871773765A",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Very well, let's go back to the house. We're going to discover things about Lil. Open her box, and we'll proceed to apply elixirs.",
        "str": "很好，我们回到房子。我们将发现有关Lil的事情。打开她的盒子，然后我们将继续施用药剂。"
    },
    {
        "key": "C81403CF4FA4C82FAED5DE8756E2CA62",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "I've given you a flashlight, which is quite important in this mode due to the heavy darkness. Pay attention to the numbers on your UI at the bottom; you can equip a maximum of 10 items, with the first slot always being your hand. You can navigate through your inventory by pressing the Previous and Next Equipment keys, or on the keyboard by using the number keys or the mouse wheel.",
        "str": "我给了你一个手电筒，在这种模式下非常重要，因为黑暗很浓重。注意底部UI上的数字；你最多可以装备10件物品，第一个槽位始终是你的手。你可以通过按上一个和下一个装备键或使用数字键或鼠标滚轮在键盘上浏览你的库存。"
    },
    {
        "key": "D4415F9B4956C597DB9E6A83027B1BA5",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Enter the house and find the dolls' room.",
        "str": "进入房子，找到人偶的房间。"
    },
    {
        "key": "DA59B2F64FE3F88E166FBB87C3277E27",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Elixirs",
        "str": "药剂"
    },
    {
        "key": "DACB1EDD4AB27C4191AE658DC5537B43",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Soul papers are mystical scrolls, extraordinarily unique, and there are two types: Soul Papers and Soul Dark Papers. In other words, the Soul Scroll and its dark version. These scrolls reveal the likes and dislikes of the dolls. We'll start with the normal scroll. Apply the scroll to the doll.",
        "str": "灵魂纸是神秘的卷轴，非常独特，有两种类型：灵魂纸和灵魂黑纸。换句话说，灵魂卷轴及其黑暗版本。这些卷轴揭示了人偶的喜好和厌恶。我们从普通卷轴开始。将卷轴应用于人偶。"
    },
    {
        "key": "DE5FC3C747F16F46862210AB8687B02C",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Remember that you won't be able to interact with anything right now since this is the tutorial. Now, let's turn on the electricity. Equip your fuse and go to the back of the house to activate it.",
        "str": "记得这只是教程，你还不能真正操作任何东西。现在，让我们开始通电吧。装上保险丝，然后去房子后面启动电源。"
    },
    {
        "key": "E0B35CB146811DDBDBF08EA339FC6413",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Good job, lastly, light the fireplace.",
        "str": "干得好，最后，点燃壁炉。"
    },
    {
        "key": "F4A0E2E14D59BC41D9DCA8958FB25D18",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Inventory",
        "str": "物品栏"
    },
    {
        "key": "FA633E4748F6A0B7063605A5816F9060",
        "location": "/Game/levels/dynamic/dynamicHouseMap.dynamicHouseMap_C:ExecuteUbergraph_dynamicHouseMap [Script Bytecode]",
        "id": "Before you go, I want to clarify that in this game mode, you can return to the refuge by driving the caravan. Lastly, what we've done with Lil in this mode will need to be done for each of the dolls to identify the impostor. Keep in mind that the campaign mode is intended to be a co-op experience in the future, which is why it's important that it isn't brief or quick. Each case could take you days to solve, but it's asynchronous, meaning you can exit whenever you want and continue playing later, and everything will remain as you left it.",
        "str": "在你离开之前，我想澄清一下，在这种游戏模式中，你可以驾驶房车返回避难所。最后，我们在这种模式中对Lil所做的事情需要对每个人偶都做，以识别冒名顶替者。请记住，未来的战役模式旨在成为一种合作体验，这就是为什么它不应该是简短或快速的。每个案件可能需要你几天的时间来解决，但它是异步的，这意味着你可以在任何时候退出，稍后继续游戏，一切都会保持你离开时的状态。"
    },
    {
        "key": "A7540C9740107C15DAA704B657E53E8D",
        "location": "/Game/levels/safeMap.safeMap:PersistentLevel.helpInformation_C_1.description",
        "id": "Doll Who is a deduction board game where two players try to guess the identity of the character chosen by their opponent. Each player has a board with a series of characters with different traits. By asking \nyes\n or \nno\n questions about the characters' features (such as whether they have a hat or appeared in the daycare map), players eliminate options until they discover the opponent's character. The first player to guess correctly wins.",
        "str": "Doll Who 是一款推理棋盘游戏，两个玩家试图猜出对手所选择的角色。每个玩家都有一个包含不同特征的角色的棋盘。通过询问关于角色特征的“是”或“否”问题（例如是否戴帽子或是否出现在托儿所地图中），玩家逐渐排除选项，直至找到对手的角色。最先猜中对手角色的玩家获胜。"
    },
    {
        "key": "7ABFF0E544D6C598A2E7A08FC98250AC",
        "location": "/Game/levels/safeMap.safeMap_C:ExecuteUbergraph_safeMap [Script Bytecode]",
        "id": "Welcome to the refuge! This is your safe zone. Here, you can start a new game on the computer, view your collection of dolls and trophies, and also customize this place by clicking on the colour palette icon on your desktop!",
        "str": "欢迎来到避难所！"
    },
    {
        "key": "D89BAF654ED277C71FA76392443B5C15",
        "location": "/Game/levels/safeMap.safeMap_C:ExecuteUbergraph_safeMap [Script Bytecode]",
        "id": "Refuge",
        "str": "避难所"
    },
    {
        "key": "04119A0C4BAB3896CA5F208DA41F582A",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "It looks like the impostor has done some events. These will be displayed on your left panel. Review them and then access the room again.",
        "str": "看起来卧底做了一些事情。这显示在左侧面板上。请在检查后再次进入房间。"
    },
    {
        "key": "08FC22B7498C0918FE50E0AB274B5889",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Remember to keep the door locked with the latch. If you don't, the dolls can come out, alter your tests, and even kill you.",
        "str": "记住要用插销锁好门。如果不这样做，人偶可能会出逃，改变测试结果，甚至杀死你。"
    },
    {
        "key": "095285C644BBCF6CF4EA8EBA3F985F9F",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "This is the dolls' room. Here you will see what they have done while you slept. Take a look at them, and then let's go back to the room.",
        "str": "这是人偶的房间。在这里，你可以看到他们在你睡觉时做了什么。看一眼他们，然后回到房间。"
    },
    {
        "key": "0A3D5EC2435134BB35A189A4B69C4101",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Room",
        "str": "房间"
    },
    {
        "key": "0B2A9AAE439C83F5A59624AAE1C48AAE",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Traps",
        "str": "陷阱"
    },
    {
        "key": "0E888CB843DF92D55BD437BEB42F99EB",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Let's go back to the panel and start weeding out suspects who don't match our evidence.",
        "str": "回到白板并剔除不符合证据的嫌疑人。"
    },
    {
        "key": "34AFF09749F832F67360E0923F227F4F",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Discards",
        "str": "排除"
    },
    {
        "key": "3A4B9B82453A3C78DD066BBB53AFC867",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "We only have 3 left. Let's go to the room to see who has moved and who has been moved.",
        "str": "只剩下3个了。去房间看看有谁动了。"
    },
    {
        "key": "47254BE24A91487FA9DB08918BAFEBED",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "You've made it through the tutorial, you know the basics! To finish the tutorial, go to bed and sleep. You will go to your safe zone. Use the computer to start a game!",
        "str": "你已完成教程，了解了游戏基础！前往床上睡觉结束教程，你将进入安全区。使用电脑即可开始游戏！"
    },
    {
        "key": "5220ED3742AAD20F53D18B86F110B811",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "You are the guardian of dolls, one of them is alive and intends to kill you. You must discover which one it is before 09:00.",
        "str": "你是人偶的看守者，其中有个人偶被恶灵附身，打算杀死你。你必须在09:00之前找出它是谁。"
    },
    {
        "key": "53A10A0B4D95DA3B8371AEA60CDFE0CF",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Perfect, we have 5 suspects left. Let's go to sleep again.",
        "str": "很好，还剩下5位嫌疑人。再去睡觉吧。"
    },
    {
        "key": "611C7265455A03D1E36711A53C897A09",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Detective",
        "str": "侦探"
    },
    {
        "key": "6621BA854D50DB36041E58AFE9A250D5",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "It's the last hour. We only have this chance to find out. Let's go to the room.",
        "str": "这是最后一小时。只剩这次机会能找出卧底了。前往房间吧。"
    },
    {
        "key": "75DB907D4C3E80ED887ACDA2AADF602C",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "You will have a detective role in which you will have to determine, based on the impostor's likes and dislikes listed on the panel to your left, what he likes and doesn't like, who his enemies, friends, loves, or hated individuals are.",
        "str": "你将扮演侦探，根据左侧面板上列出的卧底信息，确定人偶的偏好，谁是他的敌人、朋友、爱人或仇人。"
    },
    {
        "key": "7F18D69F420F68627730E39CE489A2EC",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Let's set a trap for the impostor. We already know that he likes TV and music. Turn on the lights.",
        "str": "让我们给卧底设个陷阱。我们知道他喜欢电视和音乐。打开灯吧。"
    },
    {
        "key": "815797D840D1A6AEC45401A9A3996DE0",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Don't trust",
        "str": "切勿轻信"
    },
    {
        "key": "8C5FD61743DD0F208E4EC58CE1706DB2",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Well done!",
        "str": "干得好！"
    },
    {
        "key": "8E5AFE684A85E803C6F5ADAD24BB2C1D",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Events",
        "str": "事件"
    },
    {
        "key": "AFC2B2644270E4DDA921A0AA0660BA68",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "To sleep, simply point at the bed and click on it.",
        "str": "指向并点击床即可睡觉。"
    },
    {
        "key": "B300BE9147F822906E54C7A227BF23BA",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Welcome to Doll Impostor!",
        "str": "欢迎来到《人偶卧底》！"
    },
    {
        "key": "BD7ABF40452E652BF9FDDDB2B311BA06",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "To start, there are 20 dolls. Check them on the board and when you are ready, go to the room to meet them.",
        "str": "一开始有20个人偶。先在白板上查看它们，准备好之后，就去房间与它们见面。"
    },
    {
        "key": "BDC3A1864427A6240202ACAFA646188C",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Go to sleep and let's see if the clue is solved in the next hour.",
        "str": "去睡觉吧，看看在下一个小时能否找到线索。"
    },
    {
        "key": "BF587B62432840655EC059BE5054E289",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "It is clear that he likes blood. It could be either of the two, but check the boxes. Freddy is missing and his box is full of blood. He left this verdict for you. When you are sure, discard the candidate.",
        "str": "很明显他喜欢血。可能是两者中的任何一个，但你得检查那些盒子。弗莱迪不见了，而他的盒子里满是血。当你确定后，就可以排除这个嫌疑人了。"
    },
    {
        "key": "CA75CAA04ADC71945FDEF1B6843F94B0",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "He has fallen into our trap and turned off the lights. We can now discard all those who like the light.",
        "str": "他中了我们的圈套，把灯关掉了。我们现在可以排除所有喜欢光明的人。"
    },
    {
        "key": "DA89561543E0C4540EBC6E8D9B7FCF8A",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Door",
        "str": "门"
    },
    {
        "key": "E9E818494A07397E77DFC8AEBFE671E0",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Dolls",
        "str": "人偶"
    },
    {
        "key": "FA89ED204F28013D22A23CAE49ADAC61",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "Sleep",
        "str": "睡觉"
    },
    {
        "key": "FAD096A047C791A2435446A372491A59",
        "location": "/Game/levels/tutorialMap.tutorialMap_C:ExecuteUbergraph_tutorialMap [Script Bytecode]",
        "id": "We have Ricky on the couch and Monty at the piano, so we already have the 2 possible candidates. However, we are not going to take the 50% risk. Let's go back to the room to rule out the one that can't be and wait for his next event.",
        "str": "Ricky在沙发上，Monty在钢琴旁，所以我们有了两个可能的人选。不过我们不会去冒那50%的风险。回到房间，排除那个不可能的人选，然后等待他的下一次行动，从而确认人选。"
    },
    {
        "key": "09526FC54AA5D1983C1048ACADED01C4",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "You have now accepted the case. Outside of the tutorial, you could abandon the case at any time, but you won't be able to find new cases once you've accepted. With this settled, travel to the location when you're ready.",
        "str": "你现已接手此案。除非是在教程之外，否则你随时都可以放弃，但接受后就不能再找新案件了。一切就绪后，准备好就前往目的地吧。"
    },
    {
        "key": "4D302CC441AA152588642EBA8EEA7EB1",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "Select those objects to craft another one.",
        "str": "选择那些物品来制作另一个。"
    },
    {
        "key": "73A9B2C843821751DC8E15949E11E92D",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "Welcome to Career Mode. In this game mode, you will need to uncover information about the dolls and solve each case correctly.",
        "str": "欢迎来到生涯模式。在此模式中，你需要揭示人偶的信息，并正确地解开每个案件。."
    },
    {
        "key": "7C1F780F43047787285C17A26FC3B81B",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "Once you finish the tutorial, you may face financial difficulties. To address this, you can play the classic mode, and depending on the difficulty level, it will grant you more or less money to use in the cases.",
        "str": "完成教程后，您可能会面临财务困难。为了解决这个问题，您可以玩经典模式，根据难度级别，它会给您提供更多或更少的钱用于案件中。"
    },
    {
        "key": "95529FFB4B5C7B8BF0D230810B54E147",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "Career Mode",
        "str": "生涯模式"
    },
    {
        "key": "AE1DC03A4DE0A9637C9F3FA115C15F1E",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "The rest of the players have been kicked for you to complete the tutorial.",
        "str": "其余玩家已被踢出让您完成教程."
    },
    {
        "key": "B6B8AB3240DFFCDFFC11278ACB172188",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "Review the various tabs to get familiar with them. When you're ready, click the Accept button to take on this case.",
        "str": "浏览各个标签了解内容。准备好后，点击接受按钮接手案件。"
    },
    {
        "key": "C41C767549A43DC569EB449CD5A2E3B5",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "Be careful if you perform this action you will lose all your career mode progress, returning again to the tutorial, this includes your case history.",
        "str": "请小心，如果您执行此操作，您将失去所有职业模式进度，再次返回教程，这包括您的案例历史."
    },
    {
        "key": "E8BEC2F04785EFD5186984BE67C4B88A",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "Earn Money",
        "str": "赚钱"
    },
    {
        "key": "E979BCF04F7CD32E81237BA9F94E25E1",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "I've given you some items, go to the craft section to craft an elixir, you can do it manually or directly by pressing on the elixir you want.",
        "str": "我给了你一些物品，去合成部分合成一种药剂，你可以手动合成，也可以直接按下你想要的药剂。"
    },
    {
        "key": "F6BDE5474A3477CD35632DB6FFE0323F",
        "location": "/Game/UI/career/careerUI.careerUI_C:ExecuteUbergraph_careerUI [Script Bytecode]",
        "id": "You will have a case available with various dynamic parameters. It will display information like the current one, and if you don't like it, you can search for a new case in exchange for money. Currently, in tutorial mode, we have limited your interactions; you won't be able to buy, craft, exit the case, or search for a new case.",
        "str": "您将获得一个案件，其中包含多种动态参数。它会显示类似当前案件的信息，如果不满意，您可以用钱换取新的案件。现在，在教程模式中，我们对您的操作进行了限制；您暂时无法购买、制作、退出或者寻找新案件。"
    },
    {
        "key": "074C094B4943E77D1E7501B3B9277A09",
        "location": "/Game/UI/career/careerUI.careerUI_C:GetToolTipWidget [Script Bytecode]",
        "id": "The costs will be reflected in the prices of the items in the store",
        "str": "成本将体现在商品价格上。"
    },
    {
        "key": "F21C143F425251C3614B29BD90BD710D",
        "location": "/Game/UI/career/careerUI.careerUI_C:GetToolTipWidget_0 [Script Bytecode]",
        "id": "The lower the ratio, the more information about the dolls will be released initially (from 2 to 10)",
        "str": "比率越低，初始公开的人偶信息越多（范围2-10）。"
    },
    {
        "key": "EDA5294F43DB42C2856059B83C9A6BF5",
        "location": "/Game/UI/career/careerUI.careerUI_C:GetToolTipWidget_1 [Script Bytecode]",
        "id": "This probability will be latent every time you travel; you might encounter a box that will contain things inside.",
        "str": "每次旅行时，这种概率都会潜伏在你身边；你可能会遇到一个盒子，里面装有东西。"
    },
    {
        "key": "1DFB0F2649F857BCBC918D9215167D9B",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.DefaultSpinner.Options(0).Options.Label",
        "id": "Case",
        "str": "案件"
    },
    {
        "key": "0D064E76459E35CAA5EEC1B7D99A5BD6",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.DefaultSpinner.Options(1).Options.Label",
        "id": "Store",
        "str": "商店"
    },
    {
        "key": "A97B724146EEB86D384D6A85A4F8931C",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.DefaultSpinner.Options(2).Options.Label",
        "id": "Craft",
        "str": "制作"
    },
    {
        "key": "A62242A84125B60CCB0E6EBF9F4D6E5A",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_1.Text",
        "id": "Travel",
        "str": "旅行"
    },
    {
        "key": "232AD0824B82F9E7EF1AA1B176CE4C01",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_10.Text",
        "id": "Case Number",
        "str": "案件数"
    },
    {
        "key": "B7532CB642D1303CB57FF89D855B8189",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_14.Text",
        "id": "Days elapsed",
        "str": "逝去天数"
    },
    {
        "key": "7716C8834922818B3241669F9591DF51",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_16.Text",
        "id": "Costs",
        "str": "花费"
    },
    {
        "key": "6CC9B0464D976A210322DEBEA26DF03A",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_18.Text",
        "id": "Find a new case",
        "str": "寻找新案件"
    },
    {
        "key": "CD5FB4064AF5065F802F34BE8FF95BC3",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_19.Text",
        "id": "Ratio",
        "str": "比率"
    },
    {
        "key": "1AD8A0BB4BE62F5DE96885933B1E65FE",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_2.Text",
        "id": "Drop Case",
        "str": "放弃案件"
    },
    {
        "key": "612C02664338E6F05B4191AC77E63DA9",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_23.Text",
        "id": "Recipe List",
        "str": "配方表"
    },
    {
        "key": "FF2636BB4F223BBD00D531A30CF9EB06",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_24.Text",
        "id": "Case History",
        "str": "案件历史"
    },
    {
        "key": "8634768E4D6C9A9C68755EA705B1A09E",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_25.Text",
        "id": "Box",
        "str": "盒子"
    },
    {
        "key": "8772BC8B4A8B67D8026B5D806054E488",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_27.Text",
        "id": "Sell All",
        "str": "全部出售"
    },
    {
        "key": "F5A1B20A4E970FD062976FB28CEBCEC2",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_28.Text",
        "id": "Reset Career Mode",
        "str": "重置职业模式"
    },
    {
        "key": "5A417E174DAE0F1865DA91B85DC6AFF4",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_5.Text",
        "id": "Buy All",
        "str": "全部购买"
    },
    {
        "key": "F703F19542906B2F29A58E83811DB3AD",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_7.Text",
        "id": "You don't have enough money",
        "str": "金钱不足"
    },
    {
        "key": "84B3B484489A7A0017B755B62F30FC82",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_8.Text",
        "id": "Accept Case",
        "str": "接下案件"
    },
    {
        "key": "A0A8C36D40881280C500F9A2E5AFFDAB",
        "location": "/Game/UI/career/careerUI.careerUI_C:WidgetTree.TextBlock_9.Text",
        "id": "Location",
        "str": "地点"
    },
    {
        "key": "5981A77041F835414316E0BEA0C3C8F8",
        "location": "/Game/UI/career/caseItem.caseItem_C:GetText [Script Bytecode]",
        "id": "YOU WIN",
        "str": "你赢了"
    },
    {
        "key": "CCDA7BC64A6EA8E1824143B0E488DCDA",
        "location": "/Game/UI/career/caseItem.caseItem_C:GetText [Script Bytecode]",
        "id": "Dropped",
        "str": "已丢弃"
    },
    {
        "key": "FE160B74470133E4B513D18D4566D063",
        "location": "/Game/UI/career/caseItem.caseItem_C:GetText [Script Bytecode]",
        "id": "YOU LOSE",
        "str": "你输了"
    },
    {
        "key": "B8F169E14DBF7CFF567A468DBF711F46",
        "location": "/Game/UI/career/caseItem.caseItem_C:GetText_2 [Script Bytecode]",
        "id": "{count} days",
        "str": "{count}天"
    },
    {
        "key": "D3C6D63B4A98961E44BEEB8EC2FEF2D1",
        "location": "/Game/UI/career/chestUI.chestUI_C:ExecuteUbergraph_chestUI [Script Bytecode]",
        "id": "Some of the elements you already have equipped",
        "str": "你已经装备了一些元素"
    },
    {
        "key": "E70BD9AB4A85E6A6ADCE4B8E7C2F2260",
        "location": "/Game/UI/career/chestUI.chestUI_C:ExecuteUbergraph_chestUI [Script Bytecode]",
        "id": "You can only equip a maximum of 10 items",
        "str": "最多只能装备10件物品"
    },
    {
        "key": "F4678833417F67BF4FB742B959AD7E80",
        "location": "/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock.Text",
        "id": "Only items that can be equipped will appear",
        "str": "只有可装备的物品才会出现"
    },
    {
        "key": "02384976426960372C5EAE9FE9459154",
        "location": "/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock_2.Text",
        "id": "Your inventory",
        "str": "你的库存"
    },
    {
        "key": "AE6D75704E0C22E2F8C107A8F12A0222",
        "location": "/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock_3.Text",
        "id": "Your equiped items",
        "str": "你的装备物品"
    },
    {
        "key": "F880818144C667A994BEEF9200987EF4",
        "location": "/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock_6.Text",
        "id": "Chest",
        "str": "宝箱"
    },
    {
        "key": "C5BF8D15413D2BD751F76EBE5D96E205",
        "location": "/Game/UI/career/chestUI.chestUI_C:WidgetTree.TextBlock_9.Text",
        "id": "All items placed in the chest will disappear when you exit the map.",
        "str": "当你退出地图时，放置在宝箱中的所有物品都会消失。"
    },
    {
        "key": "522202404C57B5F403ECFCA2EAAFE5A9",
        "location": "/Game/UI/confirm.Default__confirm_C.AcceptButton",
        "id": "Accept",
        "str": "接受"
    },
    {
        "key": "05AD525F49BFE5E3D0E076B33A48FCF0",
        "location": "/Game/UI/confirm.Default__confirm_C.DismissButton",
        "id": "Deny",
        "str": "拒绝"
    },
    {
        "key": "2BB11678432B1B25F353A1A7B4A35E7E",
        "location": "/Game/UI/customizer.customizer_C:WidgetTree.TextBlock.Text",
        "id": "Reset",
        "str": "重置"
    },
    {
        "key": "B084116A4B080238B7AB2A89F60A598E",
        "location": "/Game/UI/endingUI.endingUI_C:WidgetTree.TextBlock_2.Text",
        "id": "Replay",
        "str": "重玩"
    },
    {
        "key": "A2C08BE6447E2720D3051F8CA460E102",
        "location": "/Game/UI/endingUI.endingUI_C:WidgetTree.TextBlock_6.Text",
        "id": "Can you help me with a positive review?",
        "str": "能留下一篇好评吗？"
    },
    {
        "key": "798C9DA54AB7FB8B77443DA0EE442098",
        "location": "/Game/UI/endingUI.endingUI_C:WidgetTree.TextBlock_7.Text",
        "id": "Go to Safe House",
        "str": "前往安全屋"
    },
    {
        "key": "F699B0844A39D7ABCF0260916BF5F41B",
        "location": "/Game/UI/escapeMenu.escapeMenu_C:WidgetTree.TextBlock.Text",
        "id": "Invite players",
        "str": "邀请玩家"
    },
    {
        "key": "2CB05BB340AA7F8DFF600E83A23BDADD",
        "location": "/Game/UI/gameMenu.gameMenu_C:ExecuteUbergraph_gameMenu [Script Bytecode]",
        "id": "Unavailable",
        "str": "不可用"
    },
    {
        "key": "AD01C42149F1EF55342AAAB262E7D9C8",
        "location": "/Game/UI/gameMenu.gameMenu_C:ExecuteUbergraph_gameMenu [Script Bytecode]",
        "id": "This game mode is not available in multiplayer.",
        "str": "此游戏模式在多人游戏中不可用。"
    },
    {
        "key": "49B315454A07028DFEC0AD8B9D05C3FA",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.DefaultSpinner.Options(0).Options.Label",
        "id": "Global",
        "str": "全局"
    },
    {
        "key": "CB775D1F41F692F4946D738AA83BB23E",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.DefaultSpinner.Options(1).Options.Label",
        "id": "Friends",
        "str": "好友"
    },
    {
        "key": "BF4FA5364C39224BE93EC080C21FF285",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock.Text",
        "id": "Mixed dolls",
        "str": "混合人偶"
    },
    {
        "key": "EABE87674E476F817EEF85B6E5E00296",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_1.Text",
        "id": "Log Off",
        "str": "注销"
    },
    {
        "key": "7C43F9DE48DB63CDFAC71F9F14D83E23",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_10.Text",
        "id": "Only suitable for professionals, it starts at 08:00 and only has 3 effective clues, there are no photos or aids, some of the data is hidden, you have limited access to the map",
        "str": "适合专家游玩的难度，开始于08:00，只有3条有效线索，没有照片或帮助，隐藏一些数据，能探索的地图空间有限"
    },
    {
        "key": "95E8DAF7441719B3063BE7B84B85AE87",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_12.Text",
        "id": "Random",
        "str": "随机"
    },
    {
        "key": "21FD3D1D43C022720514D89AC43F3A61",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_13.Text",
        "id": "You have all the clues and all the information about the dolls, but this is completely random for each doll.",
        "str": "你拥有所有线索和人偶的所有信息，但每个人偶都是完全随机的。"
    },
    {
        "key": "CE77947D46EF18A8A49309A22EB6E699",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_15.Text",
        "id": "Stats",
        "str": "统计数据"
    },
    {
        "key": "B2073B1A4D61A580ED0CF1B3F4DC6364",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_16.Text",
        "id": "Race",
        "str": "比赛"
    },
    {
        "key": "A1CB54034B401A0E8BBD049E18CBFADD",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_17.Text",
        "id": "Random mode but in race mode with medals",
        "str": "随机模式，但可以竞速获得奖牌"
    },
    {
        "key": "F167859643FD37B593110B9A866E9EB3",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_18.Text",
        "id": "Endless",
        "str": "无尽"
    },
    {
        "key": "5A6092F448BE77B01756FFA578E7435A",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_19.Text",
        "id": "Random but endless mode, you have a 3-minute timer to fix the case and there is a leaderboard of leaders",
        "str": "随机的无尽模式，你有3分钟来解决案件。该模式有排行榜"
    },
    {
        "key": "3E3E53D64E987B22AD5967A23BB5AC57",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_2.Text",
        "id": "Extras",
        "str": "额外内容"
    },
    {
        "key": "AB6B64D64932C68A8743709693AF267B",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_20.Text",
        "id": "Position",
        "str": "位置"
    },
    {
        "key": "450F1E1F4012C5BD1CE1A88E6A7F6CC1",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_21.Text",
        "id": "Name",
        "str": "名字"
    },
    {
        "key": "901E40384D0FE3FAD3508F98408C18C7",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_22.Text",
        "id": "Nights",
        "str": "夜晚"
    },
    {
        "key": "2DEA68154A30209AA0698497E47F9DA9",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_24.Text",
        "id": "Leaderboard Endless",
        "str": "无尽排行榜"
    },
    {
        "key": "E80DB2AE4EFABFA02AFEEBA33FADDDEC",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_25.Text",
        "id": "As you unlock new levels, new ones will appear!",
        "str": "解锁新关卡后，会出现新的人偶！"
    },
    {
        "key": "C6A278F94000AE1F35ED22BF78A7ADE5",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_268.Text",
        "id": "New Game",
        "str": "新游戏"
    },
    {
        "key": "2D485E384A7ABE47FD44F58A954301B6",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_4.Text",
        "id": "Hard",
        "str": "困难"
    },
    {
        "key": "63D67BF743D308E3537D9794B417B76A",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_5.Text",
        "id": "Nightmare",
        "str": "梦魇"
    },
    {
        "key": "B1FA0BA34DA00CA6BA1EE9B4D8E18772",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_6.Text",
        "id": "One more step of difficulty, there are no extra clues, You can't see the names or list",
        "str": "更上一层的难度，没有额外线索，你看不到名字或列表"
    },
    {
        "key": "DFC327DA4616540509FBF6AA40D0458B",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_7.Text",
        "id": "Expert mode, you start from 06:00, there are no extra clues and no photographs",
        "str": "专家模式，从06:00开始，没有额外线索，也没有照片"
    },
    {
        "key": "92BFF30E4E114DDBBE7791B9CC9B70A5",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_8.Text",
        "id": "Easy",
        "str": "简单"
    },
    {
        "key": "6540E66F480EE470D56246880E1AAC02",
        "location": "/Game/UI/gameMenu.gameMenu_C:WidgetTree.TextBlock_9.Text",
        "id": "The standard difficulty for each player, all clues will be made at 08:00, you have the option to see the names of the dolls and the list of actions of the imposter",
        "str": "标准的难度，所有线索出现在08:00，你可以选择查看人偶的名字和卧底的行动列表"
    },
    {
        "key": "9D736AE2415C1B2B63950AAB6F7E242C",
        "location": "/Game/UI/intro.intro_C:ExecuteUbergraph_intro [Script Bytecode]",
        "id": "We were unable to create an online session. Please check your connection and restart the game. If you decide to continue, the game will work in offline mode.",
        "str": "我们无法创建在线会话。请检查连接并重新启动游戏。如果决定继续，游戏将以离线模式运行。"
    },
    {
        "key": "D43847EE40A03CCE1238E0A4BAB8F812",
        "location": "/Game/UI/intro.intro_C:ExecuteUbergraph_intro [Script Bytecode]",
        "id": "Translated by: {people}",
        "str": "翻译：{people}"
    },
    {
        "key": "3FD3275141C46D8A4E7B0C8B98721113",
        "location": "/Game/UI/intro.intro_C:WidgetTree.TextBlock.Text",
        "id": "Settings",
        "str": "设置"
    },
    {
        "key": "1692DE874C0C2B1FC7A6A5AD1D9C1FE9",
        "location": "/Game/UI/intro.intro_C:WidgetTree.TextBlock_1.Text",
        "id": "Exit",
        "str": "退出"
    },
    {
        "key": "6AFC77BB43DCD66B4099688FF725AB13",
        "location": "/Game/UI/intro.intro_C:WidgetTree.TextBlock_2.Text",
        "id": "Play tutorial",
        "str": "游玩教程"
    },
    {
        "key": "FEA674ED45AE2C1DDB4665A9C7A8F2F8",
        "location": "/Game/UI/intro.intro_C:WidgetTree.TextBlock_268.Text",
        "id": "Play",
        "str": "游玩"
    },
    {
        "key": "36DAFDA641D110060CAB6BBD14FE3722",
        "location": "/Game/UI/MainMenu.MainMenu_C:ExecuteUbergraph_MainMenu [Script Bytecode]",
        "id": "Continue",
        "str": "继续"
    },
    {
        "key": "4634F1C448A78E03F4AC7B94441281EB",
        "location": "/Game/UI/MainMenu.MainMenu_C:WidgetTree.ResumeGameButton.ButtonText",
        "id": "Resume Game",
        "str": "继续游戏"
    },
    {
        "key": "2349798E480EE5B45348C4B753C4BCB8",
        "location": "/Game/UI/notebook.notebook_C:WidgetTree.TextBlock_31.Text",
        "id": "Doll List",
        "str": "人偶列表"
    },
    {
        "key": "694809A74B440CDD18D181A8C6BA0434",
        "location": "/Game/UI/paperInterface.paperInterface_C:GetText_1 [Script Bytecode]",
        "id": "Enemies",
        "str": "敌人"
    },
    {
        "key": "21952B6F489AA6A8308F0591867EAC2B",
        "location": "/Game/UI/paperInterface.paperInterface_C:GetText_2 [Script Bytecode]",
        "id": "Loves",
        "str": "爱慕"
    },
    {
        "key": "F328405344EB2B4CB4D34EA75D918FD1",
        "location": "/Game/UI/paperInterface.paperInterface_C:GetText_3 [Script Bytecode]",
        "id": "Hates",
        "str": "讨厌"
    },
    {
        "key": "708EA77A4ED05FDA2A6E7D9E75FCEB5C",
        "location": "/Game/UI/paperInterface.paperInterface_C:GetText_4 [Script Bytecode]",
        "id": "Partner",
        "str": "伙伴"
    },
    {
        "key": "834E072547D98D80C120CD87A16FA4D6",
        "location": "/Game/UI/paperInterfaceDynamic.paperInterfaceDynamic_C:GetText_0 [Script Bytecode]",
        "id": "Friends",
        "str": "朋友"
    },
    {
        "key": "68AB17564B2909F81E4BDC9DA994883E",
        "location": "/Game/UI/results.results_C:ExecuteUbergraph_results [Script Bytecode]",
        "id": "I’ve created a more advanced guide that explores all aspects of Career Mode. If you need more help, I recommend you check it out; it will help you solve your future cases!",
        "str": "我创建了一个更高级的指南，探讨了职业模式的各个方面。如果你需要更多帮助，我建议你看看；这会帮助你解决未来的案件！"
    },
    {
        "key": "F66992CC4272E9FD176972AE7A324010",
        "location": "/Game/UI/results.results_C:ExecuteUbergraph_results [Script Bytecode]",
        "id": "Did you understand the tutorial well?",
        "str": "你理解教程了吗？"
    },
    {
        "key": "A0C0892943CF5FC4B208879116F465F0",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_13.Text",
        "id": "Impostor Selected",
        "str": "已选择卧底"
    },
    {
        "key": "FEEFC0A44A89E6CC7163CEB1F9C3F1B8",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_15.Text",
        "id": "XP Earned",
        "str": "获得经验"
    },
    {
        "key": "7954AA374982804CD0C829A2D5575131",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_17.Text",
        "id": "Money Earned",
        "str": "赚到的钱"
    },
    {
        "key": "871328BF4DE67CD410188ABF20504C05",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_19.Text",
        "id": "Score",
        "str": "得分"
    },
    {
        "key": "EB61DE2843D60E7D5AF23EB48A67DE8B",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_21.Text",
        "id": "Rank",
        "str": "排名"
    },
    {
        "key": "1C4788334C67D107A7BE1F931C0A749E",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_24.Text",
        "id": "Your election",
        "str": "你的选择"
    },
    {
        "key": "EDD9076E4875FB9F12D9D790EC0CE5B6",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_41.Text",
        "id": "You have lost 75% for not discovering the impostor",
        "str": "未能找出卧底，损失75%的奖励"
    },
    {
        "key": "B03E1FBA436DD65AC1BDFB9C7CED0AAE",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_42.Text",
        "id": "You have lost 90% of the entire reward due to missing photos.",
        "str": "由于缺失照片，你损失了90%的奖励。"
    },
    {
        "key": "31F30B9A42A2AFBDA2E46A82CAD31875",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_43.Text",
        "id": "Total Actions",
        "str": "总行动次数"
    },
    {
        "key": "41A4BD564E59F19BEF73778A7250E40E",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_45.Text",
        "id": "Actions completed",
        "str": "行动完成"
    },
    {
        "key": "273F474146E53E4EF26ED18EF748B092",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_47.Text",
        "id": "Results",
        "str": "结果"
    },
    {
        "key": "D637C70741EFAAB1C70592A3E73D29E3",
        "location": "/Game/UI/results.results_C:WidgetTree.TextBlock_48.Text",
        "id": "Actions left",
        "str": "剩余行动"
    },
    {
        "key": "5A9082C24056C8BFE9DE8EA9069EF292",
        "location": "/Game/UI/settings.settings_C:ExecuteUbergraph_settings [Script Bytecode]",
        "id": "For the changes to take effect, restart the game",
        "str": "要使更改生效，请重新启动游戏"
    },
    {
        "key": "D59D437D4A5623BB300F15B9DEB2DFC9",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultRadioSelectSetting.Options(1).Options.Label",
        "id": "Medium",
        "str": "中"
    },
    {
        "key": "164060454E5796D5711012B8A720788E",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultRadioSelectSetting.Options(3).Options.Label",
        "id": "Extreme",
        "str": "极高"
    },
    {
        "key": "28953C4E43697949852B609F8D0793FB",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultRadioSelectSetting_17.Options(0).Options.Label",
        "id": "Off",
        "str": "关闭"
    },
    {
        "key": "7C75165C49E7D2EBD079048C7F8B471B",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultRadioSelectSetting_17.Options(1).Options.Label",
        "id": "On",
        "str": "开"
    },
    {
        "key": "7286FEC6469CF65A5584DB896D227680",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultRadioSelectSetting_212.Options(0).Options.Label",
        "id": "Windowed",
        "str": "窗口模式"
    },
    {
        "key": "02D9E8BE4D8D66E9AA05AEBCFF6F5CFD",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultRadioSelectSetting_212.Options(1).Options.Label",
        "id": "Borderless",
        "str": "无边框"
    },
    {
        "key": "2EF2F1564FC48CCE3E64C3AC2AE2520E",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultRadioSelectSetting_212.Options(2).Options.Label",
        "id": "Fullscreen",
        "str": "全屏"
    },
    {
        "key": "B7A232F04263043B9FEA5F8F6A6CBF61",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultRadioSelectSetting_277.Options(5).Options.Label",
        "id": "Unlimited",
        "str": "无限"
    },
    {
        "key": "77DAE76A48EC13E45F2983881F23E808",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultSpinner.Options(0).Options.Label",
        "id": "Gameplay Settings",
        "str": "游戏设置"
    },
    {
        "key": "45C632DB4CE38CCF91E0E89A3C83E11B",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultSpinner.Options(1).Options.Label",
        "id": "Video Settings",
        "str": "视频设置"
    },
    {
        "key": "E1BED414427264B22F7080B7ED0D0CF9",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultSpinner.Options(2).Options.Label",
        "id": "Audio Settings",
        "str": "音频设置"
    },
    {
        "key": "3E1BC428435A0C653E4D7185A4847A91",
        "location": "/Game/UI/settings.settings_C:WidgetTree.DefaultSpinner.Options(3).Options.Label",
        "id": "Input Settings",
        "str": "输入设置"
    },
    {
        "key": "F00A91DE4FD2BF7AD2CC558E36CF0D94",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock.Text",
        "id": "Field of View",
        "str": "视野"
    },
    {
        "key": "5E0DC4AE4575000AC0171F95AD85CADA",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_1.Text",
        "id": "Depth of Field",
        "str": "景深"
    },
    {
        "key": "25672B2F428EDE34D8AE9C8193CD05D6",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_10.Text",
        "id": "Mouse Sensivity",
        "str": "相机灵敏度"
    },
    {
        "key": "C217326D42A0B8DA097ECEAC0760BAF7",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_11.Text",
        "id": "Focus FOV",
        "str": "专注视野"
    },
    {
        "key": "EE03EF0F4EEDD6D027A9A894BCB0944E",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_12.Text",
        "id": "Next Equipment",
        "str": "下一件装备"
    },
    {
        "key": "928D8B36495F7CA7451ECC8DA2DD1850",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_13.Text",
        "id": "Move Forward",
        "str": "向前移动"
    },
    {
        "key": "09CCC43B450A924B1414CEA6C17A4E5D",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_137.Text",
        "id": "Language",
        "str": "语言"
    },
    {
        "key": "3E796087492560DF199BD58F4D3A3731",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_14.Text",
        "id": "Move Backward",
        "str": "向后移动"
    },
    {
        "key": "419FBA55411C37613B78B895FC47434E",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_15.Text",
        "id": "Auto settings",
        "str": "自动设置"
    },
    {
        "key": "773B946148501068D9A5B3A814B81ECD",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_16.Text",
        "id": "Move Left",
        "str": "向左移动"
    },
    {
        "key": "411476E94611E3ECBFFCB29460596681",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_17.Text",
        "id": "Move Right",
        "str": "向右移动"
    },
    {
        "key": "D9B2458843A82DBB9843FBBB36327D9A",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_18.Text",
        "id": "Turn Right",
        "str": "右转"
    },
    {
        "key": "51AAD5804BF2737D78EA9DAB73EC9BBC",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_19.Text",
        "id": "Turn Left",
        "str": "左转"
    },
    {
        "key": "0D33F3C64097DC3B481D9DBBE348199B",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_2.Text",
        "id": "Smooth Camera",
        "str": "平滑相机"
    },
    {
        "key": "636E48DB4F79961C0BCDDF9C74A51062",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_20.Text",
        "id": "Look Up",
        "str": "向上看"
    },
    {
        "key": "BE7B6E174A4CE247B21761B7E41F34D3",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_21.Text",
        "id": "Look Down",
        "str": "向下看"
    },
    {
        "key": "C8F05C9640A64BDEEDFF738C5E065267",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_22.Text",
        "id": "Crouch",
        "str": "蹲下"
    },
    {
        "key": "175FDB6245E5ED987CC62684AC5049E4",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_23.Text",
        "id": "Run",
        "str": "奔跑"
    },
    {
        "key": "460FFB13421148C8493D69A72A81E93C",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_24.Text",
        "id": "Use",
        "str": "使用"
    },
    {
        "key": "6AA4F0964EAA8B9B6AE3B3A2B62504C1",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_25.Text",
        "id": "Focus / Zoom",
        "str": "聚焦/缩放"
    },
    {
        "key": "FA1C15B1439CEF51682B81AF479F4873",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_26.Text",
        "id": "Main Menu",
        "str": "主菜单"
    },
    {
        "key": "D8F293214EA6402451DED28B75ADA37E",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_27.Text",
        "id": "Screen Resolution",
        "str": "屏幕分辨率"
    },
    {
        "key": "878E24534B885E2AB87EACB0394A0E58",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_28.Text",
        "id": "Screen Mode",
        "str": "屏幕模式"
    },
    {
        "key": "3837591C42D92A7775C0598726DC6D31",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_29.Text",
        "id": "FPS Limit",
        "str": "帧数限制"
    },
    {
        "key": "E96A210546B4B0647B0763843052BBF8",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_3.Text",
        "id": "Head Bobbing",
        "str": "镜头晃动"
    },
    {
        "key": "31D2289342B165F56F2435A9CDCEB36A",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_30.Text",
        "id": "VSync",
        "str": "垂直同步"
    },
    {
        "key": "67E721CC4E280A5A435BA1B141936D38",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_31.Text",
        "id": "Brightness",
        "str": "亮度"
    },
    {
        "key": "E75E972A4F2C8D92EC495294816B3C8D",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_32.Text",
        "id": "Temporal Upsample",
        "str": "时间上采样"
    },
    {
        "key": "1DB3301C4E2144B2C2F0BB8C2BAD0221",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_33.Text",
        "id": "Screen Percentage",
        "str": "屏幕百分比"
    },
    {
        "key": "50CA9C1F443642D2B6B6BE9E5D1E302F",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_34.Text",
        "id": "Global Illumination Method",
        "str": "全局照明方法"
    },
    {
        "key": "267DB35B4828F6D27920F0A6E4892E5A",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_35.Text",
        "id": "Reflection Method",
        "str": "反射方法"
    },
    {
        "key": "2D10309A4AE3A7B38D67BC894C9A5B97",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_36.Text",
        "id": "View Distance Quality",
        "str": "视距质量"
    },
    {
        "key": "FC57D0154C6E70C0D8C839B6EA15B4B1",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_37.Text",
        "id": "Graphic Preset",
        "str": "图形预设"
    },
    {
        "key": "7EBE45E64B8119C2AA017590BE5B8563",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_38.Text",
        "id": "Anti Aliasing Quality",
        "str": "抗锯齿质量"
    },
    {
        "key": "99AE0DEC47710525BAF347ABFE8339AB",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_39.Text",
        "id": "Post Processing Quality",
        "str": "后期处理质量"
    },
    {
        "key": "A31A353942F325B2C7CB09AA525BB7CF",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_4.Text",
        "id": "Film Grain",
        "str": "胶片颗粒"
    },
    {
        "key": "8CAC065540B47C971DC8E0B801B69618",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_40.Text",
        "id": "Shadow Quality",
        "str": "阴影质量"
    },
    {
        "key": "9806DB6E4107A01E3979C1BBEF780A91",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_41.Text",
        "id": "Global Illumination Quality",
        "str": "全局照明质量"
    },
    {
        "key": "3BF4F9F24361A6C9205652911A0DA864",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_42.Text",
        "id": "Reflection Quality",
        "str": "反射质量"
    },
    {
        "key": "B54084CB4A265AC5B0714AB02E26FFEE",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_43.Text",
        "id": "Texture Quality",
        "str": "纹理质量"
    },
    {
        "key": "BED88CC84349A7555F6F8F8702DFC716",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_44.Text",
        "id": "Visual Effects Quality",
        "str": "视觉效果质量"
    },
    {
        "key": "84658DA54182C8A664927696DBF0C2C1",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_45.Text",
        "id": "Foliage Quality",
        "str": "植被质量"
    },
    {
        "key": "16A84C9B486F4A520142D791EC349B4F",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_46.Text",
        "id": "Shading Quality",
        "str": "阴影质量"
    },
    {
        "key": "71DA9AC944078FAD69976DAAA19A68C0",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_47.Text",
        "id": "Volumetric Fog Quality",
        "str": "体积雾质量"
    },
    {
        "key": "2DE4F4264AEB90D45B2363B38034FA9E",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_48.Text",
        "id": "Music Volume",
        "str": "音乐音量"
    },
    {
        "key": "6E900D00499A2A657789138FBA3BD69E",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_49.Text",
        "id": "Previous Equipment",
        "str": "上一件装备"
    },
    {
        "key": "FCA446264B23A96037F05F8D786E8199",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_5.Text",
        "id": "Master Volume",
        "str": "主音量"
    },
    {
        "key": "0860FF434B759F0DDD299AAD5097224D",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_50.Text",
        "id": "Reset settings",
        "str": "重置设置"
    },
    {
        "key": "8AB0454249AEF8D22FE8AD9044DC5660",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_52.Text",
        "id": "Invert Horizontal Camera",
        "str": "水平反转镜头"
    },
    {
        "key": "5E2D1B9743A106B9FAB5FABB7924B6F9",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_55.Text",
        "id": "Invert Vertical Camera",
        "str": "垂直反转镜头"
    },
    {
        "key": "91B3AF44465FA4D4F6976EA74F572B41",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_56.Text",
        "id": "Show Names",
        "str": "显示名称"
    },
    {
        "key": "A7315029485656AEDD7500947B3C4E5B",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_57.Text",
        "id": "This option will only be available on Easy difficulty.",
        "str": "此选项仅在简单难度下可用。"
    },
    {
        "key": "8BFB53E5448C16FA230515968C3ABF0A",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_58.Text",
        "id": "Gamepad Sensivity",
        "str": "游戏手柄灵敏度"
    },
    {
        "key": "6DA6B9C44E1E6E888DE71D887FCB9FFC",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_6.Text",
        "id": "Effects Volume",
        "str": "效果音量"
    },
    {
        "key": "BC06960A474C65DE361B3CB2C89626B3",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_7.Text",
        "id": "Ambient Volume",
        "str": "环境音量"
    },
    {
        "key": "7826866B435A8EE313772C8979CA261B",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_8.Text",
        "id": "Jumpscares Volume",
        "str": "惊吓音量"
    },
    {
        "key": "433A6CE84E4955044C7FB38105FA63BF",
        "location": "/Game/UI/settings.settings_C:WidgetTree.TextBlock_9.Text",
        "id": "Footsteps Volume",
        "str": "脚步声音量"
    },
    {
        "key": "E83704FB46F7B20DD3FF688CFE41B9CC",
        "location": "/Game/UI/sleepUI.sleepUI_C:WidgetTree.TextBlock_11.Text",
        "id": "Cancel sleep",
        "str": "取消睡眠"
    },
    {
        "key": "57B4EC5445C5BADF97E3D48264009545",
        "location": "/Game/UI/sleepUI.sleepUI_C:WidgetTree.TextBlock_62.Text",
        "id": "Waiting players...",
        "str": "等待玩家中……"
    },
    {
        "key": "43B95535434D97B9189B93A2A4594202",
        "location": "/Game/UI/stats.stats_C:WidgetTree.dollTop1Name_1.Text",
        "id": "Amount of Information discovered",
        "str": "发现的信息量"
    },
    {
        "key": "91E782534C65C15F2165A692F6DD50F6",
        "location": "/Game/UI/stats.stats_C:WidgetTree.dollTop1Name_2.Text",
        "id": "Products purchased",
        "str": "购买的产品"
    },
    {
        "key": "056842694B3EB37BD05F159542C5CE41",
        "location": "/Game/UI/stats.stats_C:WidgetTree.dollTop1Name_3.Text",
        "id": "Products sold",
        "str": "售出的产品"
    },
    {
        "key": "C24AF8B14A567648475D7BB6A3A92BF2",
        "location": "/Game/UI/stats.stats_C:WidgetTree.dollTop1Name_4.Text",
        "id": "Number of photographs taken of dolls",
        "str": "拍摄的娃娃照片数量"
    },
    {
        "key": "D2D6CDB64FBDE59DEFB55CA83D00110D",
        "location": "/Game/UI/stats.stats_C:WidgetTree.dollTop1Name_5.Text",
        "id": "Win cases",
        "str": "赢得案件"
    },
    {
        "key": "467D9B7144D32E320B1C6AA6D55B95B5",
        "location": "/Game/UI/stats.stats_C:WidgetTree.TextBlock.Text",
        "id": "Most won map",
        "str": "获胜最多的地图"
    },
    {
        "key": "597D11E1471FC52613B9DD81AF3F6BE1",
        "location": "/Game/UI/stats.stats_C:WidgetTree.TextBlock_1.Text",
        "id": "Most Earned Doll",
        "str": "找出次数最多的人偶"
    },
    {
        "key": "11739C0E41A20499E9C760B853C45A40",
        "location": "/Game/UI/stats.stats_C:WidgetTree.TextBlock_11.Text",
        "id": "Back",
        "str": "返回"
    },
    {
        "key": "21111B074FED432D86A1CCAE77CC81C3",
        "location": "/Game/UI/waiting.waiting_C:GetText [Script Bytecode]",
        "id": "Loading...",
        "str": "正在加载……"
    },
    {
        "key": "54288BFC4059AA21CBC614AA74C19EBE",
        "location": "/Game/UI/waiting.waiting_C:GetText [Script Bytecode]",
        "id": "Waiting players {players}",
        "str": "等待玩家{players}"
    },
    {
        "key": "4A4EFF4C40A276033B050495CB6E5179",
        "location": "/Game/UI/winDetails.winDetails_C:WidgetTree.DefaultSpinner.Options(0).Options.Label",
        "id": "World",
        "str": "世界"
    },
    {
        "key": "B383AEBA444328B5A71FF88EF36A0DBB",
        "location": "/Game/UI/winDetails.winDetails_C:WidgetTree.DefaultSpinner.Options(1).Options.Label",
        "id": "Impostor",
        "str": "卧底"
    },
    {
        "key": "E447809B421298C1BDA5E4B063D86F49",
        "location": "/Game/UI/winDetails.winDetails_C:WidgetTree.TextBlock.Text",
        "id": "Action Dolls",
        "str": "动作人偶"
    },
    {
        "key": "56EBFA974C38A9B5F563D3A75E5BF0E0",
        "location": "/Game/UI/winDetails.winDetails_C:WidgetTree.TextBlock_11.Text",
        "id": "Back",
        "str": "返回"
    },
    {
        "key": "96BE28504D901245DE1115BF00D47E10",
        "location": "/Game/UI/winDetails.winDetails_C:WidgetTree.TextBlock_4.Text",
        "id": "Action Details",
        "str": "动作细节"
    },
    {
        "key": "37F6FE7147465126D16EBCAB90C1274D",
        "location": "/Game/UI/winDetails.winDetails_C:WidgetTree.TextBlock_6.Text",
        "id": "Likes",
        "str": "喜欢"
    },
    {
        "key": "C0FE25964E732C097D287AB368FB54A4",
        "location": "/Game/UI/winDetails.winDetails_C:WidgetTree.TextBlock_7.Text",
        "id": "Dislikes",
        "str": "讨厌"
    }
]
```